# Introduction

Information in living organism commuicates along the Genomics, Transcriptomics, Proteomics and Metabolomics in Central dogma. Following such stream, we might answer certain problems in different scales from individual, population, community to ecosystem. Metabolomics (i.e., the profiling and quantitation of metabolites in body fluids) is a relatively new field of "omics" studies. Different from other omics studies, metabolomics always focused on small moleculars with much lower mass than polypeptide with single or doubled charged ions. Metabolomics studies are always performed in GC-MS, GC*GC-MS[@tian2016], LC-MS, LC-MS/MS or NMR. This workflow would only cover mass spectrometry based metabolomics or XC-MS based research.

## History

According to this book section[@kusonmano2016]:

```{r history, fig.show='hold', fig.cap='Metabolomics timeline during pre- and post-metabolomics era',echo=FALSE}
knitr::include_graphics('images/metahistory.jpg')
```

- 2000-1500 BC some traditional Chinese doctors who began to evaluate the glucose level in urine of diabetic patients using ants

- 300 BC ancient Egypt and Greece that traditionally determine the urine taste to diagnose human diseases

- 1913 Joseph John Thomson and Francis William Aston mass spectrometry 

- 1946 Felix Bloch and Edward Purcell Nuclear magnetic resonance

- late 1960s chromatographic separation technique

- 1971 Pauling’s research team "Quantitative Analysis of Urine Vapor and Breath by Gas–Liquid Partition Chromatography"

- Willmitzer and his research team pioneer group in metabolomics which suggested the promotion of the metabolomics field and its potential applications from agriculture to medicine and other related areas in the biological sciences

- 2007 Human Metabolome Project consists of databases of approximately 2500 metabolites, 1200 drugs, and 3500 food components

- post-metabolomics era high-throughput analytical techniques 

## Reviews and tutorials

Some new reviews and tutorials related to this workflow could be found in those papers[@alonso2015; @cajka2016; @lu2008; @schrimpe-rutledge2016; @townsend2016; @barnes2016; @barnes2016a].

Also I noticed more and more papers showed a bunch of data process methods as strategy for metabolomics[@watrous2017;@robbatjr.2017]. If you only need metabolomics as tools to tell your story, such stratrgy could be a quick start for you.

For software, check this review[@misra2016].

## Platform for metabolomics

### XCMS online

[XCMS online](https://xcmsonline.scripps.edu/landing_page.php?pgcontent=mainPage) is hosted by Scripps Institute. If your datasets are not large, XCMS online would be the best option for you. Recently they updated the online version to support more functions for systems biology. They use metlin and iso metlin to annotate the MS/MS data. Pathway analysis is also supported. Besides, to accelerate the process, xcms online employed stream (windows only). You could use stream to connect your instrument workstation to their server and process the data along with the data acquisition automate. They also developed apps for xcms online, but I think apps for slack would be even cooler to control the data processing.

### PRIMe

[PRIMe](http://prime.psc.riken.jp/Metabolomics_Software/) is from RIKEN and UC Davis. It supports mzML and major MS vendor formats. They defined own file format ABF and eco-system for omics studies. The software are updated almost everyday. You could use MS-DIAL for untargeted analysis and MRMOROBS for targeted analysis. For annotation, they developed MS-FINDER and statistic tools with excel. This platform could replaced the dear software from company and well prepared for MS/MS data analysis and lipidomics. They are open source, work on Windows and also could run within mathmamtics. However, they don't cover pathway analysis. Another feature is they always show the most recently spectral records from public repositories. You could always get the updated MSP spectra files for your own data analysis.

If you make GC-MS based metabolomics, this paper[@matsuo2017] could be nice start.

### OpenMS

[OpenMS](https://www.openms.de/) is another good platform for mass spectrum data analysis developed with C++. You could use them as plugin of [KNIME](https://www.knime.org/). I suggest anyone who want to be a data scientist to get familiar with platform like KNIME because they supplied various API for different programme language, which is easy to use and show every steps for others. Also TOPPView in OpenMS could be the best software to visualize the MS data. You could always use the metabolomics workflow to train starter about details in data processing. pyOpenMS and OpenSWATH are also used in this platform. If you want to turn into industry, this platform fit you best because you might get a clear idea about solution and workflow. 

### MZmine 2

[MZmine 2](http://mzmine.github.io/) has three version developed on Java platform and the lastest version is included into [MSDK](https://msdk.github.io/). Similar function could be found from MZmine 2 as shown in XCMS online. However, MZmine 2 do not have pathway analysis. You could use metaboanalyst for that purpose. Actually, you could go into MSDK to find similar function supplied by [ProteoSuite](http://www.proteosuite.org) and [Openchrom](https://www.openchrom.net/). If you are a experienced coder for Java, you should start here.

### XCMS

[xcms](https://bioconductor.org/packages/release/bioc/html/xcms.html) is different from xcms online while they might share the same code. I used it almost every data to run local metabolomics data analysis. Recently, they will change their version to xcms 3 with major update for object class. Their data format would integrate into the MSnbase package and the parameters would be easy to set up for each step. Normally, I will use msconvert-IPO-xcms-xMSannotator-metaboanalyst as workflow to process the offline data. It could accelerate the process by parallel processing. However, if you are not familiar with R, you would better to choose some software above.

### Emory MaHPIC

This platform is composed by several R packages from Emory University including [apLCMS](https://sourceforge.net/projects/aplcms/) to collect the data, [xMSanalyzer](https://sourceforge.net/projects/xmsanalyzer/) to handle automated pipeline for large-scale, non-targeted metabolomics data, [xMSannotator](https://sourceforge.net/projects/xmsannotator/) for annotation of LC-MS data and [Mummichog](https://code.google.com/archive/p/atcg/wikis/mummichog_for_metabolomics.wiki) for pathway and network analysis for high-throughput metabolomics. This platform would be preferred by someone from environmental science to study exposome. I always use xMSannotator to annotate the LC-MS data.

### Others

- [MAVEN](http://genomics-pubs.princeton.edu/mzroll/index.php?show=index) from Princeton University

- [RAMclustR](https://vpr.colostate.edu/pmf/wp-content/uploads/sites/23/2015/06/Broeckling_2014.pdf) from Colorado State University

- [MAIT](https://www.bioconductor.org/packages/release/bioc/html/MAIT.html) based on xcms

- [enviGCMS](https://github.com/yufree/enviGCMS) from me

- [Metabolights](http://www.ebi.ac.uk/metabolights/) for sharing data

## Trends in Metabolomics

```{r gtrends,eval=FALSE}
library(gtrendsR)
res <- gtrends(c("metabolomics", "metabolomics"), geo = c("CA","US"))
plot(res)
```

```{r}
library(rentrez)
papers_by_year <- function(years, search_term){
    return(sapply(years, function(y) entrez_search(db="pubmed",term=search_term, mindate=y, maxdate=y, retmax=0)$count))
}
years <- 1987:2017
total_papers <- papers_by_year(years, "")
omics <- c("genomic", "epigenomic", "metabolomics", "metagenomic", "proteomic", "transcriptomic", "pharmacogenomic", "connectomic")
trend_data <- sapply(omics, function(t) papers_by_year(years, t))
trend_props <- trend_data/total_papers
library(reshape)
library(ggplot2)
trend_df <- melt(data.frame(years, trend_data), id.vars="years")
p <- ggplot(trend_df, aes(years, value, colour=variable))
p + geom_line(size=1) + scale_y_log10("number of papers")
```

### Quantitative Metabolomics

Those papers [@kapoore2016b; @jorge2016a].

### High throughput Metabolomics

Those papers [@zampieri2017]

- Cohort size
- Temporal resolution
- Spatial resolution

## Data sharing

### Data hosting

See this paper[@haug2017]:

- [MetaboLights](http://www.ebi.ac.uk/metabolights/) EU based

- [The Metabolomics Workbench](http://www.metabolomicsworkbench.org/) US based

- [MetabolomeXchange](http://www.metabolomexchange.org/site/) search engine

- [W4M[@guitton2017]](http://workflow4metabolomics.org/)

### MS Database with annotation

#### MS/MS

- [MassBank](http://www.massbank.jp/?lang=en)

- [GNPS](https://gnps.ucsd.edu/ProteoSAFe/static/gnps-splash.jsp)

- [ReSpect](http://spectra.psc.riken.jp/): phytochemicals

- [Metlin](https://metlin.scripps.edu/)

- [LipidBlast](http://fiehnlab.ucdavis.edu/projects/LipidBlast): in silico prediction

- [MZcloud](https://www.mzcloud.org/)

- [NIST](https://www.nist.gov/srd/nist-standard-reference-database-1a-v17): Not free

#### MS

- [Fiehn Lab](http://fiehnlab.ucdavis.edu/projects/binbase-setup)

- [NIST](https://www.nist.gov/srd/nist-standard-reference-database-1a-v17): No free

### Compounds Database

- [HMDB](http://www.hmdb.ca/)

- [Lipid Maps](http://www.lipidmaps.org/)

- [KEGG](http://www.genome.jp/kegg/)

- [PubChem](https://pubchem.ncbi.nlm.nih.gov/)

- [Chemspider](http://www.chemspider.com/)

- [T3DB](http://www.t3db.ca/)

### Pathway Database

## Workflow

```{r, echo = F}
DiagrammeR::grViz("digraph workflow {
node [shape = box]
A [label = '@@1']
B [label = '@@2']
C [label = '@@3']
D [label = '@@4']
E [label = '@@5']
F [label = '@@6']
G [label = '@@7']
H [label = '@@8']
I [label = '@@9']
J [label = '@@10']
K [label = '@@11']
L [label = '@@12']
M [label = '@@13']
N [label = '@@14']
O [label = '@@15']

A -> B -> C -> D -> E -> F -> G -> H
H -> I
I -> J
H -> J -> K -> L
L -> M -> N
L -> O
                  }

[1]: 'Raw data'
[2]: 'mzxml or CDF'
[3]: 'DoE folder'
[4]: 'peaks list'
[5]: 'retention time correction'
[6]: 'peaks grouping'
[7]: 'peaks filling'
[8]: 'raw peaks'
[9]: 'data visulization'
[10]: 'batch effects correction'
[11]: 'corrected peaks'
[12]: 'annotation'
[13]: 'metabolomics pathway analysis'
[14]: 'omics analysis'
[15]: 'biomarkers discovery/diagnoise'

                  ")
```