<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Meta-Workflow</title>
  <meta name="description" content="This is a workflow for metabolomics studies.">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="Meta-Workflow" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="images/cover.png" />
  <meta property="og:description" content="This is a workflow for metabolomics studies." />
  <meta name="github-repo" content="yufree/metaworkflow" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Meta-Workflow" />
  
  <meta name="twitter:description" content="This is a workflow for metabolomics studies." />
  <meta name="twitter:image" content="images/cover.png" />

<meta name="author" content="Miao YU">


<meta name="date" content="2018-11-27">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="demo-1.html">

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.3/htmlwidgets.js"></script>
<script src="libs/viz-0.3/viz.js"></script>
<link href="libs/DiagrammeR-styles-0.2/styles.css" rel="stylesheet" />
<script src="libs/grViz-binding-1.0.0/grViz.js"></script>


<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Metabolomics Workflow</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#history"><i class="fa fa-check"></i><b>1.1</b> History</a><ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#history-of-mass-spectrometry"><i class="fa fa-check"></i><b>1.1.1</b> History of Mass Spectrometry</a></li>
<li class="chapter" data-level="1.1.2" data-path="introduction.html"><a href="introduction.html#history-of-metabolomcis"><i class="fa fa-check"></i><b>1.1.2</b> History of Metabolomcis</a></li>
<li class="chapter" data-level="1.1.3" data-path="introduction.html"><a href="introduction.html#defination"><i class="fa fa-check"></i><b>1.1.3</b> Defination</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#reviews-and-tutorials"><i class="fa fa-check"></i><b>1.2</b> Reviews and tutorials</a><ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#workflow"><i class="fa fa-check"></i><b>1.2.1</b> Workflow</a></li>
<li class="chapter" data-level="1.2.2" data-path="introduction.html"><a href="introduction.html#data-analysis"><i class="fa fa-check"></i><b>1.2.2</b> Data analysis</a></li>
<li class="chapter" data-level="1.2.3" data-path="introduction.html"><a href="introduction.html#application"><i class="fa fa-check"></i><b>1.2.3</b> Application</a></li>
<li class="chapter" data-level="1.2.4" data-path="introduction.html"><a href="introduction.html#challenge"><i class="fa fa-check"></i><b>1.2.4</b> Challenge</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#trends-in-metabolomics"><i class="fa fa-check"></i><b>1.3</b> Trends in Metabolomics</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#workflow-1"><i class="fa fa-check"></i><b>1.4</b> Workflow</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="exprimental-designdoe.html"><a href="exprimental-designdoe.html"><i class="fa fa-check"></i><b>2</b> Exprimental design(DoE)</a><ul>
<li class="chapter" data-level="2.1" data-path="exprimental-designdoe.html"><a href="exprimental-designdoe.html#software"><i class="fa fa-check"></i><b>2.1</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="pretreatment.html"><a href="pretreatment.html"><i class="fa fa-check"></i><b>3</b> Pretreatment</a><ul>
<li class="chapter" data-level="3.1" data-path="pretreatment.html"><a href="pretreatment.html#quenching"><i class="fa fa-check"></i><b>3.1</b> Quenching</a></li>
<li class="chapter" data-level="3.2" data-path="pretreatment.html"><a href="pretreatment.html#extraction"><i class="fa fa-check"></i><b>3.2</b> Extraction</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html"><i class="fa fa-check"></i><b>4</b> Instrumental analysis</a></li>
<li class="chapter" data-level="5" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html"><i class="fa fa-check"></i><b>5</b> Raw data pretreatment</a><ul>
<li class="chapter" data-level="5.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#peak-extraction"><i class="fa fa-check"></i><b>5.1</b> Peak extraction</a></li>
<li class="chapter" data-level="5.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#retention-time-correction"><i class="fa fa-check"></i><b>5.2</b> Retention Time Correction</a></li>
<li class="chapter" data-level="5.3" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#filling-missing-values"><i class="fa fa-check"></i><b>5.3</b> Filling missing values</a></li>
<li class="chapter" data-level="5.4" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#spectral-deconvolution"><i class="fa fa-check"></i><b>5.4</b> Spectral deconvolution</a></li>
<li class="chapter" data-level="5.5" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#dynamic-range"><i class="fa fa-check"></i><b>5.5</b> Dynamic Range</a><ul>
<li class="chapter" data-level="5.5.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#non-detects"><i class="fa fa-check"></i><b>5.5.1</b> Non-detects</a></li>
<li class="chapter" data-level="5.5.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#over-detection-limit"><i class="fa fa-check"></i><b>5.5.2</b> Over Detection Limit</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#software-1"><i class="fa fa-check"></i><b>5.6</b> Software</a><ul>
<li class="chapter" data-level="5.6.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#peak-picking"><i class="fa fa-check"></i><b>5.6.1</b> Peak picking</a></li>
<li class="chapter" data-level="5.6.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#for-msms"><i class="fa fa-check"></i><b>5.6.2</b> For MS/MS</a></li>
<li class="chapter" data-level="5.6.3" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#improved-peak-picking"><i class="fa fa-check"></i><b>5.6.3</b> Improved Peak picking</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="peaks-normalization.html"><a href="peaks-normalization.html"><i class="fa fa-check"></i><b>6</b> Peaks normalization</a><ul>
<li class="chapter" data-level="6.1" data-path="peaks-normalization.html"><a href="peaks-normalization.html#peak-misidentification"><i class="fa fa-check"></i><b>6.1</b> Peak misidentification</a></li>
<li class="chapter" data-level="6.2" data-path="peaks-normalization.html"><a href="peaks-normalization.html#rsd-filter"><i class="fa fa-check"></i><b>6.2</b> RSD Filter</a></li>
<li class="chapter" data-level="6.3" data-path="peaks-normalization.html"><a href="peaks-normalization.html#power-analysis-filter"><i class="fa fa-check"></i><b>6.3</b> Power Analysis Filter</a></li>
<li class="chapter" data-level="6.4" data-path="peaks-normalization.html"><a href="peaks-normalization.html#normalization"><i class="fa fa-check"></i><b>6.4</b> Normalization</a><ul>
<li class="chapter" data-level="6.4.1" data-path="peaks-normalization.html"><a href="peaks-normalization.html#unsupervised-methods"><i class="fa fa-check"></i><b>6.4.1</b> Unsupervised methods</a></li>
<li class="chapter" data-level="6.4.2" data-path="peaks-normalization.html"><a href="peaks-normalization.html#supervised-methods"><i class="fa fa-check"></i><b>6.4.2</b> Supervised methods</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="peaks-normalization.html"><a href="peaks-normalization.html#software-2"><i class="fa fa-check"></i><b>6.5</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="annotation.html"><a href="annotation.html"><i class="fa fa-check"></i><b>7</b> Annotation</a><ul>
<li class="chapter" data-level="7.1" data-path="annotation.html"><a href="annotation.html#issues-in-annotation"><i class="fa fa-check"></i><b>7.1</b> Issues in annotation</a></li>
<li class="chapter" data-level="7.2" data-path="annotation.html"><a href="annotation.html#annotation-v.s.-identification"><i class="fa fa-check"></i><b>7.2</b> Annotation v.s. identification</a></li>
<li class="chapter" data-level="7.3" data-path="annotation.html"><a href="annotation.html#ms-database-for-annotation"><i class="fa fa-check"></i><b>7.3</b> MS Database for annotation</a><ul>
<li class="chapter" data-level="7.3.1" data-path="annotation.html"><a href="annotation.html#msms"><i class="fa fa-check"></i><b>7.3.1</b> MS/MS</a></li>
<li class="chapter" data-level="7.3.2" data-path="annotation.html"><a href="annotation.html#ms"><i class="fa fa-check"></i><b>7.3.2</b> MS</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="annotation.html"><a href="annotation.html#compounds-database"><i class="fa fa-check"></i><b>7.4</b> Compounds Database</a></li>
<li class="chapter" data-level="7.5" data-path="annotation.html"><a href="annotation.html#software-3"><i class="fa fa-check"></i><b>7.5</b> Software</a><ul>
<li class="chapter" data-level="7.5.1" data-path="annotation.html"><a href="annotation.html#adducts-list"><i class="fa fa-check"></i><b>7.5.1</b> Adducts list</a></li>
<li class="chapter" data-level="7.5.2" data-path="annotation.html"><a href="annotation.html#molgen"><i class="fa fa-check"></i><b>7.5.2</b> Molgen</a></li>
<li class="chapter" data-level="7.5.3" data-path="annotation.html"><a href="annotation.html#isotope"><i class="fa fa-check"></i><b>7.5.3</b> Isotope</a></li>
<li class="chapter" data-level="7.5.4" data-path="annotation.html"><a href="annotation.html#mffinder"><i class="fa fa-check"></i><b>7.5.4</b> mfFinder</a></li>
<li class="chapter" data-level="7.5.5" data-path="annotation.html"><a href="annotation.html#camera"><i class="fa fa-check"></i><b>7.5.5</b> CAMERA</a></li>
<li class="chapter" data-level="7.5.6" data-path="annotation.html"><a href="annotation.html#ramclustr"><i class="fa fa-check"></i><b>7.5.6</b> RAMClustR</a></li>
<li class="chapter" data-level="7.5.7" data-path="annotation.html"><a href="annotation.html#pmd"><i class="fa fa-check"></i><b>7.5.7</b> pmd</a></li>
<li class="chapter" data-level="7.5.8" data-path="annotation.html"><a href="annotation.html#nontarget"><i class="fa fa-check"></i><b>7.5.8</b> nontarget</a></li>
<li class="chapter" data-level="7.5.9" data-path="annotation.html"><a href="annotation.html#xmsannotator"><i class="fa fa-check"></i><b>7.5.9</b> xMSannotator</a></li>
<li class="chapter" data-level="7.5.10" data-path="annotation.html"><a href="annotation.html#cfm-id"><i class="fa fa-check"></i><b>7.5.10</b> CFM-ID</a></li>
<li class="chapter" data-level="7.5.11" data-path="annotation.html"><a href="annotation.html#mine"><i class="fa fa-check"></i><b>7.5.11</b> MINE</a></li>
<li class="chapter" data-level="7.5.12" data-path="annotation.html"><a href="annotation.html#interpretmsspectrum"><i class="fa fa-check"></i><b>7.5.12</b> InterpretMSSpectrum</a></li>
<li class="chapter" data-level="7.5.13" data-path="annotation.html"><a href="annotation.html#for-ident"><i class="fa fa-check"></i><b>7.5.13</b> For Ident</a></li>
<li class="chapter" data-level="7.5.14" data-path="annotation.html"><a href="annotation.html#mzmatch"><i class="fa fa-check"></i><b>7.5.14</b> mzmatch</a></li>
<li class="chapter" data-level="7.5.15" data-path="annotation.html"><a href="annotation.html#mz.unity"><i class="fa fa-check"></i><b>7.5.15</b> mz.unity</a></li>
<li class="chapter" data-level="7.5.16" data-path="annotation.html"><a href="annotation.html#mait"><i class="fa fa-check"></i><b>7.5.16</b> MAIT</a></li>
<li class="chapter" data-level="7.5.17" data-path="annotation.html"><a href="annotation.html#probmetab"><i class="fa fa-check"></i><b>7.5.17</b> ProbMetab</a></li>
<li class="chapter" data-level="7.5.18" data-path="annotation.html"><a href="annotation.html#ramsi"><i class="fa fa-check"></i><b>7.5.18</b> RAMSI</a></li>
<li class="chapter" data-level="7.5.19" data-path="annotation.html"><a href="annotation.html#sirius"><i class="fa fa-check"></i><b>7.5.19</b> Sirius</a></li>
<li class="chapter" data-level="7.5.20" data-path="annotation.html"><a href="annotation.html#mi-pack"><i class="fa fa-check"></i><b>7.5.20</b> MI-Pack</a></li>
<li class="chapter" data-level="7.5.21" data-path="annotation.html"><a href="annotation.html#plantmat"><i class="fa fa-check"></i><b>7.5.21</b> Plantmat</a></li>
<li class="chapter" data-level="7.5.22" data-path="annotation.html"><a href="annotation.html#metfamily"><i class="fa fa-check"></i><b>7.5.22</b> MetFamily</a></li>
<li class="chapter" data-level="7.5.23" data-path="annotation.html"><a href="annotation.html#lipidmatch"><i class="fa fa-check"></i><b>7.5.23</b> Lipidmatch</a></li>
<li class="chapter" data-level="7.5.24" data-path="annotation.html"><a href="annotation.html#molfind"><i class="fa fa-check"></i><b>7.5.24</b> MolFind</a></li>
<li class="chapter" data-level="7.5.25" data-path="annotation.html"><a href="annotation.html#metfusion"><i class="fa fa-check"></i><b>7.5.25</b> MetFusion</a></li>
<li class="chapter" data-level="7.5.26" data-path="annotation.html"><a href="annotation.html#imet"><i class="fa fa-check"></i><b>7.5.26</b> iMet</a></li>
<li class="chapter" data-level="7.5.27" data-path="annotation.html"><a href="annotation.html#metscape"><i class="fa fa-check"></i><b>7.5.27</b> Metscape</a></li>
<li class="chapter" data-level="7.5.28" data-path="annotation.html"><a href="annotation.html#metfrag"><i class="fa fa-check"></i><b>7.5.28</b> MetFrag</a></li>
<li class="chapter" data-level="7.5.29" data-path="annotation.html"><a href="annotation.html#lipidfrag"><i class="fa fa-check"></i><b>7.5.29</b> LipidFrag</a></li>
<li class="chapter" data-level="7.5.30" data-path="annotation.html"><a href="annotation.html#mycompoundid"><i class="fa fa-check"></i><b>7.5.30</b> MycompoundID</a></li>
<li class="chapter" data-level="7.5.31" data-path="annotation.html"><a href="annotation.html#magma"><i class="fa fa-check"></i><b>7.5.31</b> magma</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="annotation.html"><a href="annotation.html#ms-dia"><i class="fa fa-check"></i><b>7.6</b> MS-DIA</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="omics-analysis.html"><a href="omics-analysis.html"><i class="fa fa-check"></i><b>8</b> Omics analysis</a><ul>
<li class="chapter" data-level="8.1" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-analysis"><i class="fa fa-check"></i><b>8.1</b> Pathway analysis</a><ul>
<li class="chapter" data-level="8.1.1" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-database"><i class="fa fa-check"></i><b>8.1.1</b> Pathway Database</a></li>
<li class="chapter" data-level="8.1.2" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-software"><i class="fa fa-check"></i><b>8.1.2</b> Pathway software</a></li>
</ul></li>
<li class="chapter" data-level="8.2" data-path="omics-analysis.html"><a href="omics-analysis.html#network-analysis"><i class="fa fa-check"></i><b>8.2</b> Network analysis</a></li>
<li class="chapter" data-level="8.3" data-path="omics-analysis.html"><a href="omics-analysis.html#omics-integration"><i class="fa fa-check"></i><b>8.3</b> Omics integration</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html"><i class="fa fa-check"></i><b>9</b> Common analysis methods for metabolomics</a><ul>
<li class="chapter" data-level="9.1" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#basic-statistical-analysis"><i class="fa fa-check"></i><b>9.1</b> Basic Statistical Analysis</a></li>
<li class="chapter" data-level="9.2" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#differences-analysis"><i class="fa fa-check"></i><b>9.2</b> Differences analysis</a></li>
<li class="chapter" data-level="9.3" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#pca"><i class="fa fa-check"></i><b>9.3</b> PCA</a></li>
<li class="chapter" data-level="9.4" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#cluster-analysis"><i class="fa fa-check"></i><b>9.4</b> Cluster Analysis</a></li>
<li class="chapter" data-level="9.5" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#plsda"><i class="fa fa-check"></i><b>9.5</b> PLSDA</a></li>
<li class="chapter" data-level="9.6" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#self-organizing-map"><i class="fa fa-check"></i><b>9.6</b> Self-organizing map</a></li>
<li class="chapter" data-level="9.7" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#canonical-correlation-analysis"><i class="fa fa-check"></i><b>9.7</b> Canonical correlation analysis</a></li>
<li class="chapter" data-level="9.8" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#software-4"><i class="fa fa-check"></i><b>9.8</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="workflow-2.html"><a href="workflow-2.html"><i class="fa fa-check"></i><b>10</b> Workflow</a><ul>
<li class="chapter" data-level="10.1" data-path="workflow-2.html"><a href="workflow-2.html#platform-for-metabolomics"><i class="fa fa-check"></i><b>10.1</b> Platform for metabolomics</a><ul>
<li class="chapter" data-level="10.1.1" data-path="workflow-2.html"><a href="workflow-2.html#xcms-online"><i class="fa fa-check"></i><b>10.1.1</b> XCMS online</a></li>
<li class="chapter" data-level="10.1.2" data-path="workflow-2.html"><a href="workflow-2.html#prime"><i class="fa fa-check"></i><b>10.1.2</b> PRIMe</a></li>
<li class="chapter" data-level="10.1.3" data-path="workflow-2.html"><a href="workflow-2.html#openms"><i class="fa fa-check"></i><b>10.1.3</b> OpenMS</a></li>
<li class="chapter" data-level="10.1.4" data-path="workflow-2.html"><a href="workflow-2.html#mzmine-2"><i class="fa fa-check"></i><b>10.1.4</b> MZmine 2</a></li>
<li class="chapter" data-level="10.1.5" data-path="workflow-2.html"><a href="workflow-2.html#xcms"><i class="fa fa-check"></i><b>10.1.5</b> XCMS</a></li>
<li class="chapter" data-level="10.1.6" data-path="workflow-2.html"><a href="workflow-2.html#emory-mahpic"><i class="fa fa-check"></i><b>10.1.6</b> Emory MaHPIC</a></li>
<li class="chapter" data-level="10.1.7" data-path="workflow-2.html"><a href="workflow-2.html#dia-data-analysis"><i class="fa fa-check"></i><b>10.1.7</b> DIA data analysis</a></li>
<li class="chapter" data-level="10.1.8" data-path="workflow-2.html"><a href="workflow-2.html#others"><i class="fa fa-check"></i><b>10.1.8</b> Others</a></li>
</ul></li>
<li class="chapter" data-level="10.2" data-path="workflow-2.html"><a href="workflow-2.html#data-sharing"><i class="fa fa-check"></i><b>10.2</b> Data sharing</a></li>
<li class="chapter" data-level="10.3" data-path="workflow-2.html"><a href="workflow-2.html#contest"><i class="fa fa-check"></i><b>10.3</b> Contest</a></li>
<li class="chapter" data-level="10.4" data-path="workflow-2.html"><a href="workflow-2.html#demo"><i class="fa fa-check"></i><b>10.4</b> Demo</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="demo-1.html"><a href="demo-1.html"><i class="fa fa-check"></i><b>11</b> Demo</a><ul>
<li class="chapter" data-level="11.1" data-path="demo-1.html"><a href="demo-1.html#project-setup"><i class="fa fa-check"></i><b>11.1</b> Project Setup</a></li>
<li class="chapter" data-level="11.2" data-path="demo-1.html"><a href="demo-1.html#data-input"><i class="fa fa-check"></i><b>11.2</b> Data input</a></li>
<li class="chapter" data-level="11.3" data-path="demo-1.html"><a href="demo-1.html#optimization"><i class="fa fa-check"></i><b>11.3</b> Optimization</a></li>
<li class="chapter" data-level="11.4" data-path="demo-1.html"><a href="demo-1.html#wrap-function"><i class="fa fa-check"></i><b>11.4</b> Wrap function</a><ul>
<li class="chapter" data-level="11.4.1" data-path="demo-1.html"><a href="demo-1.html#peak-picking-1"><i class="fa fa-check"></i><b>11.4.1</b> Peak picking</a></li>
<li class="chapter" data-level="11.4.2" data-path="demo-1.html"><a href="demo-1.html#data-correction"><i class="fa fa-check"></i><b>11.4.2</b> Data correction</a></li>
<li class="chapter" data-level="11.4.3" data-path="demo-1.html"><a href="demo-1.html#peaks-filling"><i class="fa fa-check"></i><b>11.4.3</b> Peaks filling</a></li>
</ul></li>
<li class="chapter" data-level="11.5" data-path="demo-1.html"><a href="demo-1.html#peaks-list"><i class="fa fa-check"></i><b>11.5</b> Peaks list</a></li>
<li class="chapter" data-level="11.6" data-path="demo-1.html"><a href="demo-1.html#peaks-filtering"><i class="fa fa-check"></i><b>11.6</b> Peaks filtering</a></li>
<li class="chapter" data-level="11.7" data-path="demo-1.html"><a href="demo-1.html#normalization-optional"><i class="fa fa-check"></i><b>11.7</b> Normalization (Optional)</a></li>
<li class="chapter" data-level="11.8" data-path="demo-1.html"><a href="demo-1.html#statistic-analysis"><i class="fa fa-check"></i><b>11.8</b> Statistic analysis</a></li>
<li class="chapter" data-level="11.9" data-path="demo-1.html"><a href="demo-1.html#annotation-1"><i class="fa fa-check"></i><b>11.9</b> Annotation</a></li>
<li class="chapter" data-level="11.10" data-path="demo-1.html"><a href="demo-1.html#pathway-analysis-1"><i class="fa fa-check"></i><b>11.10</b> Pathway Analysis</a></li>
<li class="chapter" data-level="11.11" data-path="demo-1.html"><a href="demo-1.html#metaboanalyst"><i class="fa fa-check"></i><b>11.11</b> MetaboAnalyst</a></li>
<li class="chapter" data-level="11.12" data-path="demo-1.html"><a href="demo-1.html#summary"><i class="fa fa-check"></i><b>11.12</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="exposome.html"><a href="exposome.html"><i class="fa fa-check"></i><b>12</b> Exposome</a><ul>
<li class="chapter" data-level="12.1" data-path="exposome.html"><a href="exposome.html#internal-exposure"><i class="fa fa-check"></i><b>12.1</b> Internal exposure</a></li>
<li class="chapter" data-level="12.2" data-path="exposome.html"><a href="exposome.html#external-exposure"><i class="fa fa-check"></i><b>12.2</b> External exposure</a><ul>
<li class="chapter" data-level="12.2.1" data-path="exposome.html"><a href="exposome.html#environmental-fate-of-compounds"><i class="fa fa-check"></i><b>12.2.1</b> Environmental fate of compounds</a></li>
<li class="chapter" data-level="12.2.2" data-path="exposome.html"><a href="exposome.html#exposure-study-database"><i class="fa fa-check"></i><b>12.2.2</b> Exposure study database</a></li>
</ul></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/yufree/metaworkflow" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Meta-Workflow</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="exposome" class="section level1">
<h1><span class="header-section-number">Chapter 12</span> Exposome</h1>
<p>According to <a href="https://www.cdc.gov/niosh/topics/exposome/default.html">CDC</a>, The exposome can be defined as the measure of all the exposures of an individual in a lifetime and how those exposures relate to health. Exposomics is the study of the exposome and relies on the application of internal and external exposure assessment methods.</p>
<ul>
<li><p>Internal exposure relies on fields of study such as genomics, metabolomics, lipidomics, transcriptomics and proteomics.</p></li>
<li><p>External exposure assessment relies on measuring environmental stressors.</p></li>
</ul>
<div id="internal-exposure" class="section level2">
<h2><span class="header-section-number">12.1</span> Internal exposure</h2>
<ul>
<li><p><a href="http://www.hmdb.ca/">HMDB</a> is a freely available electronic database containing detailed information about small molecule metabolites found in the human body.</p></li>
<li><p><a href="http://www.lipidmaps.org/">Lipid Maps</a></p></li>
<li><p><a href="https://jcggdb.jp/rcmg/glycodb/Ms_ResultSearch">GMDB</a> a multistage tandem mass spectral database using a variety of structurally defined glycans.</p></li>
<li><p><a href="https://www.genome.jp/kegg/compound/">KEGG</a> is a collection of small molecules, biopolymers, and other chemical substances that are relevant to biological systems.</p></li>
<li><p><a href="https://www.vmh.life/">Virtual Metabolic Human Database</a> integrating human and gut microbiome metabolism with nutrition and disease.</p></li>
</ul>
</div>
<div id="external-exposure" class="section level2">
<h2><span class="header-section-number">12.2</span> External exposure</h2>
<div id="environmental-fate-of-compounds" class="section level3">
<h3><span class="header-section-number">12.2.1</span> Environmental fate of compounds</h3>
<div id="qspr" class="section level4">
<h4><span class="header-section-number">12.2.1.1</span> QSPR</h4>
<ul>
<li><p><a href="https://chemicalize.com/">Chemicalize</a> is a powerful online platform for chemical calculations, search, and text processing.</p></li>
<li><p><a href="http://www.moleculardescriptors.eu/resources/resources.htm">QSPR molecular descriptor generate tools list</a></p></li>
<li><p><a href="http://www.archemcalc.com/sparc.html">Spark</a> uses computational algorithms based on fundamental chemical structure theory to estimate a wide variety of reactivity parameters strictly from molecular structure.</p></li>
</ul>
<p>LogP is important for analytical chemistry. Mannhold <span class="citation">(Mannhold et al. <a href="#ref-mannhold2009">2009</a>)</span> report a comprehensive comparison of logP algorithms. Later, <a href="http://blog.rguha.net/?p=896">Rajarshi Guha</a> make a comparison with logP algorithms with CDK based on logPstar dataset. Commercial software such as Spark, ACS Labs and ChemAxon might always claim a better performance on in-house dataset compared with public software like KowWIN within EPI Suite. However, we should be careful to evaluate the influnce of logP accuracy on the metabolites or unknown compounds.</p>
</div>
<div id="fate" class="section level4">
<h4><span class="header-section-number">12.2.1.2</span> Fate</h4>
<ul>
<li><p><a href="https://www.utsc.utoronto.ca/labs/wania/downloads/">Wania Group</a> developed software tools to address various aspects of organic contaminant fate and behaviour.</p></li>
<li><p><a href="http://www.trentu.ca/academic/aminss/envmodel/models/models.html">Trent University</a> release models to predict environmental fate for pollutions such as Level 3.</p></li>
<li><p><a href="http://eawag-bbd.ethz.ch/index.html">EAWAG-BBD</a> could provide information on microbial enzyme-catalyzed reactions that are important for biotechnology.</p></li>
</ul>
</div>
</div>
<div id="exposure-study-database" class="section level3">
<h3><span class="header-section-number">12.2.2</span> Exposure study database</h3>
<ul>
<li><p>The information system <a href="https://www.pangaea.de">PANGAEA</a> is operated as an Open Access library aimed at archiving, publishing and distributing georeferenced data from earth system research.</p></li>
<li><p><a href="http://www.inchem.org/pages/ehc.html">Environmental Health Criteria (EHC) Monographs</a></p></li>
<li><p><a href="http://ctdbase.org/">CTD</a> is a robust, publicly available database that aims to advance understanding about how environmental exposures affect human health.</p></li>
<li><p><a href="https://www.mass.gov/orgs/office-of-data-management-and-outcomes-assessment">ODMOA</a> facilitates and coordinates the collection, access to, and use of public health data in order to monitor and improve population health. This data is better for general public health research for Massachusetts.</p></li>
<li><p><a href="https://seer.cancer.gov/">The Surveillance, Epidemiology, and End Results (SEER)</a> Program provides information on cancer statistics in an effort to reduce the cancer burden among the U.S. population.</p></li>
<li><p><a href="https://comptox.epa.gov/dashboard">CompTox</a> compounds, exposure and toxicity database. <a href="https://www.epa.gov/chemical-research/downloadable-computational-toxicology-data">Here</a> is related data.</p></li>
<li><p><a href="http://www.t3db.ca/">T3DB</a> is a unique bioinformatics resource that combines detailed toxin data with comprehensive toxin target information.</p></li>
<li><p><a href="http://foodb.ca/">FooDB</a> is the world’s largest and most comprehensive resource on food constituents, chemistry and biology.</p></li>
<li><p><a href="http://phenol-explorer.eu">Phenol explorer</a> is the first comprehensive database on polyphenol content in foods.</p></li>
<li><p><a href="https://www.drugbank.ca/releases/latest">Drugbank</a> is a unique bioinformatics and cheminformatics resource that combines detailed drug data with comprehensive drug target information.</p></li>
<li><p><a href="http://lmdb.ca">LMDB</a> is a freely available electronic database containing detailed information about small molecule metabolites found in different livestock species.</p></li>
</ul>

<div id="refs" class="references">
<div>
<p>Aguilar-Mogas, Antoni, Marta Sales-Pardo, Miriam Navarro, Roger Guimerà, and Oscar Yanes. 2017. “IMet: A Network-Based Computational Tool To Assist in the Annotation of Metabolites from Tandem Mass Spectra.” <em>Anal. Chem.</em> 89 (6): 3474–82. doi:<a href="https://doi.org/10.1021/acs.analchem.6b04512">10.1021/acs.analchem.6b04512</a>.</p>
</div>
<div>
<p>Allam-Ndoul, Bénédicte, Frédéric Guénard, Véronique Garneau, Hubert Cormier, Olivier Barbier, Louis Pérusse, and Marie-Claude Vohl. 2016. “Association Between Metabolite Profiles, Metabolic Syndrome and Obesity Status.” <em>Nutrients</em> 8 (6): 324. doi:<a href="https://doi.org/10.3390/nu8060324">10.3390/nu8060324</a>.</p>
</div>
<div>
<p>Allen, Felicity, Allison Pon, Michael Wilson, Russ Greiner, and David Wishart. 2014. “CFM-ID: A Web Server for Annotation, Spectrum Prediction and Metabolite Identification from Tandem Mass Spectra.” <em>Nucleic Acids Res</em> 42 (W1): W94–W99. doi:<a href="https://doi.org/10.1093/nar/gku436">10.1093/nar/gku436</a>.</p>
</div>
<div>
<p>Alonso, Arnald, Sara Marsal, and Antonio Julià. 2015. “Analytical Methods in Untargeted Metabolomics: State of the Art in 2015.” <em>Front Bioeng Biotechnol</em> 3 (March). doi:<a href="https://doi.org/10.3389/fbioe.2015.00023">10.3389/fbioe.2015.00023</a>.</p>
</div>
<div>
<p>Andersson, Martin. 2009. “A Comparison of Nine PLS1 Algorithms.” <em>J. Chemom.</em> 23 (10): 518–29. doi:<a href="https://doi.org/10.1002/cem.1248">10.1002/cem.1248</a>.</p>
</div>
<div>
<p>B. Dunn, Warwick, David I. Broadhurst, Helen J. Atherton, Royston Goodacre, and Julian L. Griffin. 2011. “Systems Level Studies of Mammalian Metabolomes: The Roles of Mass Spectrometry and Nuclear Magnetic Resonance Spectroscopy.” <em>Chem. Soc. Rev.</em> 40 (1): 387–426. doi:<a href="https://doi.org/10.1039/B906712B">10.1039/B906712B</a>.</p>
</div>
<div>
<p>Baran, Richard, and Trent R. Northen. 2013. “Robust Automated Mass Spectra Interpretation and Chemical Formula Calculation Using Mixed Integer Linear Programming.” <em>Anal. Chem.</em> 85 (20): 9777–84. doi:<a href="https://doi.org/10.1021/ac402180c">10.1021/ac402180c</a>.</p>
</div>
<div>
<p>Barnes, Stephen, H. Paul Benton, Krista Casazza, Sara J. Cooper, Xiangqin Cui, Xiuxia Du, Jeffrey Engler, et al. 2016a. “Training in Metabolomics Research. I. Designing the Experiment, Collecting and Extracting Samples and Generating Metabolomics Data.” <em>J. Mass Spectrom.</em> 51 (7): 461–75. doi:<a href="https://doi.org/10.1002/jms.3782">10.1002/jms.3782</a>.</p>
</div>
<div>
<p>———. 2016b. “Training in Metabolomics Research. II. Processing and Statistical Analysis of Metabolomics Data, Metabolite Identification, Pathway Analysis, Applications of Metabolomics and Its Future.” <em>J. Mass Spectrom.</em> 51 (8): 535–48. doi:<a href="https://doi.org/10.1002/jms.3780">10.1002/jms.3780</a>.</p>
</div>
<div>
<p>Basu, Sumanta, William Duren, Charles R. Evans, Charles F. Burant, George Michailidis, and Alla Karnovsky. 2017. “Sparse Network Modeling and Metscape-Based Visualization Methods for the Analysis of Large-Scale Metabolomics Data.” <em>Bioinformatics</em> 33 (10): 1545–53. doi:<a href="https://doi.org/10.1093/bioinformatics/btx012">10.1093/bioinformatics/btx012</a>.</p>
</div>
<div>
<p>Begou, O., H. G. Gika, I. D. Wilson, and G. Theodoridis. 2017. “Hyphenated MS-Based Targeted Approaches in Metabolomics.” <em>Analyst</em> 142 (17): 3079–3100. doi:<a href="https://doi.org/10.1039/C7AN00812K">10.1039/C7AN00812K</a>.</p>
</div>
<div>
<p>Bennett, Bryson D., Elizabeth H. Kimball, Melissa Gao, Robin Osterhout, Stephen J. Van Dien, and Joshua D. Rabinowitz. 2009. “Absolute Metabolite Concentrations and Implied Enzyme Active Site Occupancy in Escherichia Coli.” <em>Nat Chem Biol</em> 5 (8): 593–99. doi:<a href="https://doi.org/10.1038/nchembio.186">10.1038/nchembio.186</a>.</p>
</div>
<div>
<p>Bilbao, Aivett, Emmanuel Varesio, Jeremy Luban, Caterina Strambio-De-Castillia, Gérard Hopfgartner, Markus Müller, and Frédérique Lisacek. 2015. “Processing Strategies and Software Solutions for Data-Independent Acquisition in Mass Spectrometry.” <em>PROTEOMICS</em> 15 (5-6): 964–80. doi:<a href="https://doi.org/10.1002/pmic.201400323">10.1002/pmic.201400323</a>.</p>
</div>
<div>
<p>Blaise, Benjamin J., Gonçalo Correia, Adrienne Tin, J. Hunter Young, Anne-Claire Vergnaud, Matthew Lewis, Jake T. M. Pearce, et al. 2016. “Power Analysis and Sample Size Determination in Metabolic Phenotyping.” <em>Anal. Chem.</em> 88 (10): 5179–88. doi:<a href="https://doi.org/10.1021/acs.analchem.6b00188">10.1021/acs.analchem.6b00188</a>.</p>
</div>
<div>
<p>Brereton, Richard G., and Gavin R. Lloyd. 2018. “Partial Least Squares Discriminant Analysis for Chemometrics and Metabolomics: How Scores, Loadings, and Weights Differ According to Two Common Algorithms.” <em>J. Chemom.</em> 32 (4): e3028. doi:<a href="https://doi.org/10.1002/cem.3028">10.1002/cem.3028</a>.</p>
</div>
<div>
<p>Broeckling, C. D., F. A. Afsar, S. Neumann, A. Ben-Hur, and J. E. Prenni. 2014. “RAMClust: A Novel Feature Clustering Method Enables Spectral-Matching-Based Annotation for Metabolomics Data.” <em>Anal. Chem.</em> 86 (14): 6812–7. doi:<a href="https://doi.org/10.1021/ac501530d">10.1021/ac501530d</a>.</p>
</div>
<div>
<p>Bundy, Jacob G., Matthew P. Davey, and Mark R. Viant. 2009. “Environmental Metabolomics: A Critical Review and Future Perspectives.” <em>Metabolomics</em> 5 (1): 3. doi:<a href="https://doi.org/10.1007/s11306-008-0152-0">10.1007/s11306-008-0152-0</a>.</p>
</div>
<div>
<p>Cai, Qingpo, Jessica A. Alvarez, Jian Kang, and Tianwei Yu. 2017. “Network Marker Selection for Untargeted LCMS Metabolomics Data.” <em>J. Proteome Res.</em> 16 (3): 1261–9. doi:<a href="https://doi.org/10.1021/acs.jproteome.6b00861">10.1021/acs.jproteome.6b00861</a>.</p>
</div>
<div>
<p>Cajka, Tomas, and Oliver Fiehn. 2016. “Toward Merging Untargeted and Targeted Methods in Mass Spectrometry-Based Metabolomics and Lipidomics.” <em>Anal. Chem.</em> 88 (1): 524–45. doi:<a href="https://doi.org/10.1021/acs.analchem.5b04491">10.1021/acs.analchem.5b04491</a>.</p>
</div>
<div>
<p>Castro-Puyana, María, Raquel Pérez-Míguez, Lidia Montero, and Miguel Herrero. 2017. “Application of Mass Spectrometry-Based Metabolomics Approaches for Food Safety, Quality and Traceability.” <em>TrAC Trends in Analytical Chemistry</em> 93 (August): 102–18. doi:<a href="https://doi.org/10.1016/j.trac.2017.05.004">10.1016/j.trac.2017.05.004</a>.</p>
</div>
<div>
<p>Chambers, Matthew C., Brendan Maclean, Robert Burke, Dario Amodei, Daniel L. Ruderman, Steffen Neumann, Laurent Gatto, et al. 2012. “A Cross-Platform Toolkit for Mass Spectrometry and Proteomics.” <em>Nat. Biotechnol.</em> 30 (October): 918–20. doi:<a href="https://doi.org/10.1038/nbt.2377">10.1038/nbt.2377</a>.</p>
</div>
<div>
<p>Chang, Hui-Yin, Ching-Tai Chen, T. Mamie Lih, Ke-Shiuan Lynn, Chiun-Gung Juo, Wen-Lian Hsu, and Ting-Yi Sung. 2016. “IMet-Q: A User-Friendly Tool for Label-Free Metabolomics Quantitation Using Dynamic Peak-Width Determination.” <em>PLOS ONE</em> 11 (1): e0146112. doi:<a href="https://doi.org/10.1371/journal.pone.0146112">10.1371/journal.pone.0146112</a>.</p>
</div>
<div>
<p>Chen, Gengbo, Scott Walmsley, Gemmy C. M. Cheung, Liyan Chen, Ching-Yu Cheng, Roger W. Beuerman, Tien Yin Wong, Lei Zhou, and Hyungwon Choi. 2017. “Customized Consensus Spectral Library Building for Untargeted Quantitative Metabolomics Analysis with Data Independent Acquisition Mass Spectrometry and MetaboDIA Workflow.” <em>Anal. Chem.</em> 89 (9): 4897–4906. doi:<a href="https://doi.org/10.1021/acs.analchem.6b05006">10.1021/acs.analchem.6b05006</a>.</p>
</div>
<div>
<p>Cheng, Susan, Svati H. Shah, Elizabeth J. Corwin, Oliver Fiehn, Robert L. Fitzgerald, Robert E. Gerszten, Thomas Illig, et al. 2017. “Potential Impact and Study Considerations of Metabolomics in Cardiovascular Health and Disease: A Scientific Statement From the American Heart Association.” <em>Circ. Cardiovasc. Genet.</em> 10 (2): e000032. doi:<a href="https://doi.org/10.1161/HCG.0000000000000032">10.1161/HCG.0000000000000032</a>.</p>
</div>
<div>
<p>Creek, Darren J., Andris Jankevics, Karl E. V. Burgess, Rainer Breitling, and Michael P. Barrett. 2012. “IDEOM: An Excel Interface for Analysis of LCMS-Based Metabolomics Data.” <em>Bioinformatics</em> 28 (7): 1048–9. doi:<a href="https://doi.org/10.1093/bioinformatics/bts069">10.1093/bioinformatics/bts069</a>.</p>
</div>
<div>
<p>De Livera, Alysha M., Daniel A. Dias, David De Souza, Thusitha Rupasinghe, James Pyke, Dedreia Tull, Ute Roessner, Malcolm McConville, and Terence P. Speed. 2012. “Normalizing and Integrating Metabolomics Data.” <em>Anal. Chem.</em> 84 (24): 10768–76. doi:<a href="https://doi.org/10.1021/ac302748b">10.1021/ac302748b</a>.</p>
</div>
<div>
<p>DeFelice, Brian C., Sajjan Singh Mehta, Stephanie Samra, Tomáš Čajka, Benjamin Wancewicz, Johannes F. Fahrmann, and Oliver Fiehn. 2017. “Mass Spectral Feature List Optimizer (MS-FLO): A Tool To Minimize False Positive Peak Reports in Untargeted Liquid ChromatographyMass Spectroscopy (LC-MS) Data Processing.” <em>Anal. Chem.</em> 89 (6): 3250–5. doi:<a href="https://doi.org/10.1021/acs.analchem.6b04372">10.1021/acs.analchem.6b04372</a>.</p>
</div>
<div>
<p>Domingo-Almenara, Xavier, Jesus Brezmes, Maria Vinaixa, Sara Samino, Noelia Ramirez, Marta Ramon-Krauel, Carles Lerin, et al. 2016. “ERah: A Computational Tool Integrating Spectral Deconvolution and Alignment with Quantification and Identification of Metabolites in GC/MS-Based Metabolomics.” <em>Anal. Chem.</em> 88 (19): 9821–9. doi:<a href="https://doi.org/10.1021/acs.analchem.6b02927">10.1021/acs.analchem.6b02927</a>.</p>
</div>
<div>
<p>Domingo-Almenara, Xavier, J. Rafael Montenegro-Burke, H. Paul Benton, and Gary Siuzdak. 2018. “Annotation: A Computational Solution for Streamlining Metabolomics Analysis.” <em>Anal. Chem.</em> 90 (1): 480–89. doi:<a href="https://doi.org/10.1021/acs.analchem.7b03929">10.1021/acs.analchem.7b03929</a>.</p>
</div>
<div>
<p>Du, Xiuxia, and Steven H Zeisel. 2013. “SPECTRAL DECONVOLUTION FOR GAS CHROMATOGRAPHY MASS SPECTROMETRY-BASED METABOLOMICS: CURRENT STATUS AND FUTURE PERSPECTIVES.” <em>Computational and Structural Biotechnology Journal</em> 4 (5): 1–10. doi:<a href="https://doi.org/10.5936/csbj.201301013">10.5936/csbj.201301013</a>.</p>
</div>
<div>
<p>Dudzik, Danuta, Cecilia Barbas-Bernardos, Antonia García, and Coral Barbas. 2018. “Quality Assurance Procedures for Mass Spectrometry Untargeted Metabolomics. a Review.” <em>Journal of Pharmaceutical and Biomedical Analysis</em>, Review issue 2017, 147 (January): 149–73. doi:<a href="https://doi.org/10.1016/j.jpba.2017.07.044">10.1016/j.jpba.2017.07.044</a>.</p>
</div>
<div>
<p>Dührkop, Kai, Huibin Shen, Marvin Meusel, Juho Rousu, and Sebastian Böcker. 2015. “Searching Molecular Structure Databases with Tandem Mass Spectra Using CSI:FingerID.” <em>PNAS</em> 112 (41): 12580–5. doi:<a href="https://doi.org/10.1073/pnas.1509788112">10.1073/pnas.1509788112</a>.</p>
</div>
<div>
<p>Fenaille, François, Pierre Barbier Saint-Hilaire, Kathleen Rousseau, and Christophe Junot. 2017. “Data Acquisition Workflows in Liquid Chromatography Coupled to High Resolution Mass Spectrometry-Based Metabolomics: Where Do We Stand?” <em>Journal of Chromatography A</em> 1526 (Supplement C): 1–12. doi:<a href="https://doi.org/10.1016/j.chroma.2017.10.043">10.1016/j.chroma.2017.10.043</a>.</p>
</div>
<div>
<p>Fernández-Albert, Francesc, Rafael Llorach, Cristina Andrés-Lacueva, and Alexandre Perera. 2014. “An R Package to Analyse LC/MS Metabolomic Data: MAIT (Metabolite Automatic Identification Toolkit).” <em>Bioinformatics</em> 30 (13): 1937–9. doi:<a href="https://doi.org/10.1093/bioinformatics/btu136">10.1093/bioinformatics/btu136</a>.</p>
</div>
<div>
<p>Fessenden, Marissa. 2016. “Metabolomics: Small Molecules, Single Cells.” <em>Nature</em> 540 (7631): 153–55. doi:<a href="https://doi.org/10.1038/540153a">10.1038/540153a</a>.</p>
</div>
<div>
<p>Fiehn, Oliver. 2002. “Metabolomics the Link Between Genotypes and Phenotypes.” <em>Plant Mol Biol</em> 48 (1): 155–71. doi:<a href="https://doi.org/10.1023/A:1013713905833">10.1023/A:1013713905833</a>.</p>
</div>
<div>
<p>Franceschi, Pietro, Domenico Masuero, Urska Vrhovsek, Fulvio Mattivi, and Ron Wehrens. 2012. “A Benchmark Spike-in Data Set for Biomarker Identification in Metabolomics.” <em>J. Chemometrics</em> 26 (1-2): 16–24. doi:<a href="https://doi.org/10.1002/cem.1420">10.1002/cem.1420</a>.</p>
</div>
<div>
<p>Fu, Hai-Yan, Ou Hu, Yue-Ming Zhang, Li Zhang, Jing-Jing Song, Peang Lu, Qing-Xia Zheng, et al. 2017. “Mass-Spectra-Based Peak Alignment for Automatic Nontargeted Metabolic Profiling Analysis for Biomarker Screening in Plant Samples.” <em>Journal of Chromatography A</em> 1513 (Supplement C): 201–9. doi:<a href="https://doi.org/10.1016/j.chroma.2017.07.044">10.1016/j.chroma.2017.07.044</a>.</p>
</div>
<div>
<p>Gerlich, Michael, and Steffen Neumann. 2013. “MetFusion: Integration of Compound Identification Strategies.” <em>J. Mass Spectrom.</em> 48 (3): 291–98. doi:<a href="https://doi.org/10.1002/jms.3123">10.1002/jms.3123</a>.</p>
</div>
<div>
<p>Gika, Helen G., Georgios A. Theodoridis, Robert S. Plumb, and Ian D. Wilson. 2014. “Current Practice of Liquid ChromatographyMass Spectrometry in Metabolomics and Metabonomics.” <em>Journal of Pharmaceutical and Biomedical Analysis</em>, Review papers on pharmaceutical and biomedical analysis 2013, 87 (January): 12–25. doi:<a href="https://doi.org/10.1016/j.jpba.2013.06.032">10.1016/j.jpba.2013.06.032</a>.</p>
</div>
<div>
<p>Goldansaz, Seyed Ali, An Chi Guo, Tanvir Sajed, Michael A. Steele, Graham S. Plastow, and David S. Wishart. 2017. “Livestock Metabolomics and the Livestock Metabolome: A Systematic Review.” <em>PLOS ONE</em> 12 (5): e0177675. doi:<a href="https://doi.org/10.1371/journal.pone.0177675">10.1371/journal.pone.0177675</a>.</p>
</div>
<div>
<p>Gonzalez-Riano, Carolina, Antonia Garcia, and Coral Barbas. 2016. “Metabolomics Studies in Brain Tissue: A Review.” <em>Journal of Pharmaceutical and Biomedical Analysis</em>, Review issue 2016, 130 (October): 141–68. doi:<a href="https://doi.org/10.1016/j.jpba.2016.07.008">10.1016/j.jpba.2016.07.008</a>.</p>
</div>
<div>
<p>Griffiths, William J., Therese Koal, Yuqin Wang, Matthias Kohl, David P. Enot, and Hans-Peter Deigner. 2010. “Targeted Metabolomics for Biomarker Discovery.” <em>Angew. Chem. Int. Ed.</em> 49 (32): 5426–45. doi:<a href="https://doi.org/10.1002/anie.200905579">10.1002/anie.200905579</a>.</p>
</div>
<div>
<p>Gromski, Piotr S., Howbeer Muhamadali, David I. Ellis, Yun Xu, Elon Correa, Michael L. Turner, and Royston Goodacre. 2015. “A Tutorial Review: Metabolomics and Partial Least Squares-Discriminant Analysis a Marriage of Convenience or a Shotgun Wedding.” <em>Analytica Chimica Acta</em> 879 (June): 10–23. doi:<a href="https://doi.org/10.1016/j.aca.2015.02.012">10.1016/j.aca.2015.02.012</a>.</p>
</div>
<div>
<p>Guijas, Carlos, J. Rafael Montenegro-Burke, Xavier Domingo-Almenara, Amelia Palermo, Benedikt Warth, Gerrit Hermann, Gunda Koellensperger, et al. 2018. “METLIN: A Technology Platform for Identifying Knowns and Unknowns.” <em>Anal. Chem.</em> 90 (5): 3156–64. doi:<a href="https://doi.org/10.1021/acs.analchem.7b04424">10.1021/acs.analchem.7b04424</a>.</p>
</div>
<div>
<p>Guitton, Yann, Marie Tremblay-Franco, Gildas Le Corguillé, Jean-François Martin, Mélanie Pétéra, Pierrick Roger-Mele, Alexis Delabrière, et al. 2017. “Create, Run, Share, Publish, and Reference Your LCMS, FIAMS, GCMS, and NMR Data Analysis Workflows with the Workflow4Metabolomics 3.0 Galaxy Online Infrastructure for Metabolomics.” <em>The International Journal of Biochemistry &amp; Cell Biology</em> 93 (Supplement C): 89–101. doi:<a href="https://doi.org/10.1016/j.biocel.2017.07.002">10.1016/j.biocel.2017.07.002</a>.</p>
</div>
<div>
<p>Haug, Kenneth, Reza M Salek, and Christoph Steinbeck. 2017. “Global Open Data Management in Metabolomics.” <em>Current Opinion in Chemical Biology</em>, Omics, 36 (February): 58–63. doi:<a href="https://doi.org/10.1016/j.cbpa.2016.12.024">10.1016/j.cbpa.2016.12.024</a>.</p>
</div>
<div>
<p>Hocher, Berthold, and Jerzy Adamski. 2017. “Metabolomics for Clinical Use and Research in Chronic Kidney Disease.” <em>Nat. Rev. Nephrol.</em> 13 (5): 269–84. doi:<a href="https://doi.org/10.1038/nrneph.2017.30">10.1038/nrneph.2017.30</a>.</p>
</div>
<div>
<p>Huang, Xiaojing, Ying-Jr Chen, Kevin Cho, Igor Nikolskiy, Peter A. Crawford, and Gary J. Patti. 2014. “X13CMS: Global Tracking of Isotopic Labels in Untargeted Metabolomics.” <em>Anal. Chem.</em> 86 (3): 1632–9. doi:<a href="https://doi.org/10.1021/ac403384n">10.1021/ac403384n</a>.</p>
</div>
<div>
<p>Hufsky, Franziska, Kerstin Scheubert, and Sebastian Böcker. 2014. “Computational Mass Spectrometry for Small-Molecule Fragmentation.” <em>TrAC Trends in Analytical Chemistry</em> 53 (January): 41–48. doi:<a href="https://doi.org/10.1016/j.trac.2013.09.008">10.1016/j.trac.2013.09.008</a>.</p>
</div>
<div>
<p>Jones, Dean P., Youngja Park, and Thomas R. Ziegler. 2012. “Nutritional Metabolomics: Progress in Addressing Complexity in Diet and Health.” <em>Annu. Rev. Nutr.</em> 32 (1): 183–202. doi:<a href="https://doi.org/10.1146/annurev-nutr-072610-145159">10.1146/annurev-nutr-072610-145159</a>.</p>
</div>
<div>
<p>Jorge, Tiago F., Ana T. Mata, and Carla António. 2016. “Mass Spectrometry as a Quantitative Tool in Plant Metabolomics.” <em>Phil. Trans. R. Soc. A</em> 374 (2079): 20150370. doi:<a href="https://doi.org/10.1098/rsta.2015.0370">10.1098/rsta.2015.0370</a>.</p>
</div>
<div>
<p>Jr, Stephen Salerno, Mahya Mehrmohamadi, Maria V. Liberti, Muting Wan, Martin T. Wells, James G. Booth, and Jason W. Locasale. 2017. “RRmix: A Method for Simultaneous Batch Effect Correction and Analysis of Metabolomics Data in the Absence of Internal Standards.” <em>PLOS ONE</em> 12 (6): e0179530. doi:<a href="https://doi.org/10.1371/journal.pone.0179530">10.1371/journal.pone.0179530</a>.</p>
</div>
<div>
<p>Kapoore, Rahul Vijay, and Seetharaman Vaidyanathan. 2016. “Towards Quantitative Mass Spectrometry-Based Metabolomics in Microbial and Mammalian Systems.” <em>Phil. Trans. R. Soc. A</em> 374 (2079): 20150363. doi:<a href="https://doi.org/10.1098/rsta.2015.0363">10.1098/rsta.2015.0363</a>.</p>
</div>
<div>
<p>Karpievitch, Yuliya V., Sonja B. Nikolic, Richard Wilson, James E. Sharman, and Lindsay M. Edwards. 2014. “Metabolomics Data Normalization with EigenMS.” <em>PLOS ONE</em> 9 (12): e116221. doi:<a href="https://doi.org/10.1371/journal.pone.0116221">10.1371/journal.pone.0116221</a>.</p>
</div>
<div>
<p>Kew, William, John W. T. Blackburn, David J. Clarke, and Dušan Uhrín. 2017. “Interactive van Krevelen Diagrams Visualisation of Mass Spectrometry Data of Complex Mixtures.” <em>Rapid Commun. Mass Spectrom.</em> 31 (7): 658–62. doi:<a href="https://doi.org/10.1002/rcm.7823">10.1002/rcm.7823</a>.</p>
</div>
<div>
<p>Koelmel, Jeremy P., Nicholas M. Kroeger, Candice Z. Ulmer, John A. Bowden, Rainey E. Patterson, Jason A. Cochran, Christopher W. W. Beecher, Timothy J. Garrett, and Richard A. Yost. 2017. “LipidMatch: An Automated Workflow for Rule-Based Lipid Identification Using Untargeted High-Resolution Tandem Mass Spectrometry Data.” <em>BMC Bioinformatics</em> 18 (July): 331. doi:<a href="https://doi.org/10.1186/s12859-017-1744-3">10.1186/s12859-017-1744-3</a>.</p>
</div>
<div>
<p>Kuhl, Carsten, Ralf Tautenhahn, Christoph Böttcher, Tony R. Larson, and Steffen Neumann. 2012. “CAMERA: An Integrated Strategy for Compound Spectra Extraction and Annotation of Liquid Chromatography/Mass Spectrometry Data Sets.” <em>Anal. Chem.</em> 84 (1): 283–89. doi:<a href="https://doi.org/10.1021/ac202450g">10.1021/ac202450g</a>.</p>
</div>
<div>
<p>Kuligowski, Julia, Ángel Sánchez-Illana, Daniel Sanjuán-Herráez, Máximo Vento, and Guillermo Quintás. 2015. “Intra-Batch Effect Correction in Liquid Chromatography-Mass Spectrometry Using Quality Control Samples and Support Vector Regression (QC-SVRC).” <em>Analyst</em> 140 (22): 7810–7. doi:<a href="https://doi.org/10.1039/C5AN01638J">10.1039/C5AN01638J</a>.</p>
</div>
<div>
<p>Kusonmano, Kanthida, Wanwipa Vongsangnak, and Pramote Chumnanpuen. 2016. “Informatics for Metabolomics.” In <em>Translational Biomedical Informatics</em>, 91–115. Advances in Experimental Medicine and Biology. Springer, Singapore. doi:<a href="https://doi.org/10.1007/978-981-10-1503-8_5">10.1007/978-981-10-1503-8_5</a>.</p>
</div>
<div>
<p>Lawson, Thomas N., Ralf J. M. Weber, Martin R. Jones, Andrew J. Chetwynd, Giovanny Rodriguez-Blanco, Riccardo Di Guida, Mark R. Viant, and Warwick B. Dunn. 2017. “MsPurity: Automated Evaluation of Precursor Ion Purity for Mass Spectrometry-Based Fragmentation in Metabolomics.” <em>Anal. Chem.</em> 89 (4): 2432–9. doi:<a href="https://doi.org/10.1021/acs.analchem.6b04358">10.1021/acs.analchem.6b04358</a>.</p>
</div>
<div>
<p>Leek, Jeffrey T., and John D. Storey. 2007. “Capturing Heterogeneity in Gene Expression Studies by Surrogate Variable Analysis.” <em>PLOS Genet</em> 3 (9): e161. doi:<a href="https://doi.org/10.1371/journal.pgen.0030161">10.1371/journal.pgen.0030161</a>.</p>
</div>
<div>
<p>———. 2008. “A General Framework for Multiple Testing Dependence.” <em>PNAS</em> 105 (48): 18718–23. doi:<a href="https://doi.org/10.1073/pnas.0808709105">10.1073/pnas.0808709105</a>.</p>
</div>
<div>
<p>Leek, Jeffrey T., W. Evan Johnson, Hilary S. Parker, Andrew E. Jaffe, and John D. Storey. 2012. “The Sva Package for Removing Batch Effects and Other Unwanted Variation in High-Throughput Experiments.” <em>Bioinformatics</em> 28 (6): 882–83. doi:<a href="https://doi.org/10.1093/bioinformatics/bts034">10.1093/bioinformatics/bts034</a>.</p>
</div>
<div>
<p>Lê Cao, Kim-Anh, Simon Boitard, and Philippe Besse. 2011. “Sparse PLS Discriminant Analysis: Biologically Relevant Feature Selection and Graphical Displays for Multiclass Problems.” <em>BMC Bioinformatics</em> 12 (June): 253. doi:<a href="https://doi.org/10.1186/1471-2105-12-253">10.1186/1471-2105-12-253</a>.</p>
</div>
<div>
<p>Li, Bo, Jing Tang, Qingxia Yang, Shuang Li, Xuejiao Cui, Yinghong Li, Yuzong Chen, Weiwei Xue, Xiaofeng Li, and Feng Zhu. 2017. “NOREVA: Normalization and Evaluation of MS-Based Metabolomics Data.” <em>Nucleic Acids Res</em> 45 (W1): W162–W170. doi:<a href="https://doi.org/10.1093/nar/gkx449">10.1093/nar/gkx449</a>.</p>
</div>
<div>
<p>Li, Liang, Ronghong Li, Jianjun Zhou, Azeret Zuniga, Avalyn E. Stanislaus, Yiman Wu, Tao Huan, et al. 2013. “MyCompoundID: Using an Evidence-Based Metabolome Library for Metabolite Identification.” <em>Anal. Chem.</em> 85 (6): 3401–8. doi:<a href="https://doi.org/10.1021/ac400099b">10.1021/ac400099b</a>.</p>
</div>
<div>
<p>Libiseller, Gunnar, Michaela Dvorzak, Ulrike Kleb, Edgar Gander, Tobias Eisenberg, Frank Madeo, Steffen Neumann, et al. 2015. “IPO: A Tool for Automated Optimization of XCMS Parameters.” <em>BMC Bioinformatics</em> 16 (April): 118. doi:<a href="https://doi.org/10.1186/s12859-015-0562-8">10.1186/s12859-015-0562-8</a>.</p>
</div>
<div>
<p>Lisec, Jan, Friederike Hoffmann, Clemens Schmitt, and Carsten Jaeger. 2016. “Extending the Dynamic Range in Metabolomics Experiments by Automatic Correction of Peaks Exceeding the Detection Limit.” <em>Anal. Chem.</em> 88 (15): 7487–92. doi:<a href="https://doi.org/10.1021/acs.analchem.6b02515">10.1021/acs.analchem.6b02515</a>.</p>
</div>
<div>
<p>Livera, Alysha M. De, Marko Sysi-Aho, Laurent Jacob, Johann A. Gagnon-Bartsch, Sandra Castillo, Julie A. Simpson, and Terence P. Speed. 2015. “Statistical Methods for Handling Unwanted Variation in Metabolomics Data.” <em>Anal. Chem.</em> 87 (7): 3606–15. doi:<a href="https://doi.org/10.1021/ac502439y">10.1021/ac502439y</a>.</p>
</div>
<div>
<p>Lu, Wenyun, Bryson D. Bennett, and Joshua D. Rabinowitz. 2008. “Analytical Strategies for LCMS-Based Targeted Metabolomics.” <em>Journal of Chromatography B</em>, Hyphenated techniques for global metabolite profiling, 871 (2): 236–42. doi:<a href="https://doi.org/10.1016/j.jchromb.2008.04.031">10.1016/j.jchromb.2008.04.031</a>.</p>
</div>
<div>
<p>Lu, Wenyun, Xiaoyang Su, Matthias S. Klein, Ian A. Lewis, Oliver Fiehn, and Joshua D. Rabinowitz. 2017. “Metabolite Measurement: Pitfalls to Avoid and Practices to Follow.” <em>Annu. Rev. Biochem.</em> 86 (1): 277–304. doi:<a href="https://doi.org/10.1146/annurev-biochem-061516-044952">10.1146/annurev-biochem-061516-044952</a>.</p>
</div>
<div>
<p>Lu, Xin, and Guowang Xu. 2008. “LC-MS Metabonomics Methodology in Biomarker Discovery.” In <em>Biomarker Methods in Drug Discovery and Development</em>, edited by Feng Wang, 291–315. Methods in Pharmacology and Toxicology. Humana Press. doi:<a href="https://doi.org/10.1007/978-1-59745-463-6_14">10.1007/978-1-59745-463-6_14</a>.</p>
</div>
<div>
<p>Luo, Xian, and Liang Li. 2017. “Metabolomics of Small Numbers of Cells: Metabolomic Profiling of 100, 1000, and 10000 Human Breast Cancer Cells.” <em>Anal. Chem.</em> 89 (21): 11664–71. doi:<a href="https://doi.org/10.1021/acs.analchem.7b03100">10.1021/acs.analchem.7b03100</a>.</p>
</div>
<div>
<p>Madsen, Rasmus, Torbjörn Lundstedt, and Johan Trygg. 2010. “Chemometrics in MetabolomicsA Review in Human Disease Diagnosis.” <em>Analytica Chimica Acta</em> 659 (1): 23–33. doi:<a href="https://doi.org/10.1016/j.aca.2009.11.042">10.1016/j.aca.2009.11.042</a>.</p>
</div>
<div>
<p>Mahieu, Nathaniel G., Jonathan L. Spalding, and Gary J. Patti. 2016. “Warpgroup: Increased Precision of Metabolomic Data Processing by Consensus Integration Bound Analysis.” <em>Bioinformatics</em> 32 (2): 268–75. doi:<a href="https://doi.org/10.1093/bioinformatics/btv564">10.1093/bioinformatics/btv564</a>.</p>
</div>
<div>
<p>Mahieu, Nathaniel G., Jonathan L. Spalding, Susan J. Gelman, and Gary J. Patti. 2016. “Defining and Detecting Complex Peak Relationships in Mass Spectral Data: The Mz.Unity Algorithm.” <em>Anal. Chem.</em> 88 (18): 9037–46. doi:<a href="https://doi.org/10.1021/acs.analchem.6b01702">10.1021/acs.analchem.6b01702</a>.</p>
</div>
<div>
<p>Mannhold, Raimund, Gennadiy I. Poda, Claude Ostermann, and Igor V. Tetko. 2009. “Calculation of Molecular Lipophilicity: State-of-the-Art and Comparison of LogP Methods on More Than 96,000 Compounds.” <em>Journal of Pharmaceutical Sciences</em> 98 (3): 861–93. doi:<a href="https://doi.org/10.1002/jps.21494">10.1002/jps.21494</a>.</p>
</div>
<div>
<p>Matsuo, Teruko, Hiroshi Tsugawa, Hiromi Miyagawa, and Eiichiro Fukusaki. 2017. “Integrated Strategy for Unknown EIMS Identification Using Quality Control Calibration Curve, Multivariate Analysis, EIMS Spectral Database, and Retention Index Prediction.” <em>Anal. Chem.</em> 89 (12): 6766–73. doi:<a href="https://doi.org/10.1021/acs.analchem.7b01010">10.1021/acs.analchem.7b01010</a>.</p>
</div>
<div>
<p>Melamud, Eugene, Livia Vastag, and Joshua D. Rabinowitz. 2010. “Metabolomic Analysis and Visualization Engine for LC-MS Data.” <em>Anal. Chem.</em> 82 (23): 9818–26. doi:<a href="https://doi.org/10.1021/ac1021166">10.1021/ac1021166</a>.</p>
</div>
<div>
<p>Menikarachchi, Lochana C., Shannon Cawley, Dennis W. Hill, L. Mark Hall, Lowell Hall, Steven Lai, Janine Wilder, and David F. Grant. 2012. “MolFind: A Software Package Enabling HPLC/MS-Based Identification of Unknown Chemical Structures.” <em>Anal. Chem.</em> 84 (21): 9388–94. doi:<a href="https://doi.org/10.1021/ac302048x">10.1021/ac302048x</a>.</p>
</div>
<div>
<p>Misra, Biswapriya B. 2018. “New Tools and Resources in Metabolomics: 20162017.” <em>ELECTROPHORESIS</em> 39 (7): 909–23. doi:<a href="https://doi.org/10.1002/elps.201700441">10.1002/elps.201700441</a>.</p>
</div>
<div>
<p>Misra, Biswapriya B., and prefix=van der family=Hooft given=Justin J. J. 2016. “Updates in Metabolomics Tools and Resources: 20142015.” <em>ELECTROPHORESIS</em> 37 (1): 86–110. doi:<a href="https://doi.org/10.1002/elps.201500417">10.1002/elps.201500417</a>.</p>
</div>
<div>
<p>Misra, Biswapriya B., Johannes F. Fahrmann, and Dmitry Grapov. 2017. “Review of Emerging Metabolomic Tools and Resources: 20152016.” <em>ELECTROPHORESIS</em> 38 (18): 2257–74. doi:<a href="https://doi.org/10.1002/elps.201700110">10.1002/elps.201700110</a>.</p>
</div>
<div>
<p>Najdekr, Lukáš, David Friedecký, Ralf Tautenhahn, Tomáš Pluskal, Junhua Wang, Yingying Huang, and Tomáš Adam. 2016. “Influence of Mass Resolving Power in Orbital Ion-Trap Mass Spectrometry-Based Metabolomics.” <em>Anal. Chem.</em> 88 (23): 11429–35. doi:<a href="https://doi.org/10.1021/acs.analchem.6b02319">10.1021/acs.analchem.6b02319</a>.</p>
</div>
<div>
<p>Ni, Yan, Mingming Su, Yunping Qiu, Wei Jia, and Xiuxia Du. 2016. “ADAP-GC 3.0: Improved Peak Detection and Deconvolution of Co-Eluting Metabolites from GC/TOF-MS Data for Metabolomics Studies.” <em>Anal. Chem.</em> 88 (17): 8802–11. doi:<a href="https://doi.org/10.1021/acs.analchem.6b02222">10.1021/acs.analchem.6b02222</a>.</p>
</div>
<div>
<p>Nikolskiy, Igor, Nathaniel G. Mahieu, Ying-Jr Chen, Ralf Tautenhahn, and Gary J. Patti. 2013. “An Untargeted Metabolomic Workflow to Improve Structural Characterization of Metabolites.” <em>Anal. Chem.</em> 85 (16): 7713–9. doi:<a href="https://doi.org/10.1021/ac400751j">10.1021/ac400751j</a>.</p>
</div>
<div>
<p>Ortmayr, Karin, Verena Charwat, Cornelia Kasper, Stephan Hann, and Gunda Koellensperger. 2016. “Uncertainty Budgeting in Fold Change Determination and Implications for Non-Targeted Metabolomics Studies in Model Systems” 142 (1): 80–90. doi:<a href="https://doi.org/10.1039/C6AN01342B">10.1039/C6AN01342B</a>.</p>
</div>
<div>
<p>Pluskal, Tomáš, Sandra Castillo, Alejandro Villar-Briones, and Matej Orešič. 2010. “MZmine 2: Modular Framework for Processing, Visualizing, and Analyzing Mass Spectrometry-Based Molecular Profile Data.” <em>BMC Bioinformatics</em> 11: 395. doi:<a href="https://doi.org/10.1186/1471-2105-11-395">10.1186/1471-2105-11-395</a>.</p>
</div>
<div>
<p>Qiu, Feng, Dennis D. Fine, Daniel J. Wherritt, Zhentian Lei, and Lloyd W. Sumner. 2016. “PlantMAT: A Metabolomics Tool for Predicting the Specialized Metabolic Potential of a System and for Large-Scale Metabolite Identifications.” <em>Anal. Chem.</em> 88 (23): 11373–83. doi:<a href="https://doi.org/10.1021/acs.analchem.6b00906">10.1021/acs.analchem.6b00906</a>.</p>
</div>
<div>
<p>Röst, Hannes L., Timo Sachsenberg, Stephan Aiche, Chris Bielow, Hendrik Weisser, Fabian Aicheler, Sandro Andreotti, et al. 2016. “OpenMS: A Flexible Open-Source Software Platform for Mass Spectrometry Data Analysis.” <em>Nat Meth</em> 13 (9): 741–48. doi:<a href="https://doi.org/10.1038/nmeth.3959">10.1038/nmeth.3959</a>.</p>
</div>
<div>
<p>Ruttkies, Christoph, Emma L. Schymanski, Sebastian Wolf, Juliane Hollender, and Steffen Neumann. 2016. “MetFrag Relaunched: Incorporating Strategies Beyond in Silico Fragmentation.” <em>Journal of Cheminformatics</em> 8 (January): 3. doi:<a href="https://doi.org/10.1186/s13321-016-0115-9">10.1186/s13321-016-0115-9</a>.</p>
</div>
<div>
<p>Silva, Ricardo R., Fabien Jourdan, Diego M. Salvanha, Fabien Letisse, Emilien L. Jamin, Simone Guidetti-Gonzalez, Carlos A. Labate, and Ricardo Z. N. Vêncio. 2014. “ProbMetab: An R Package for Bayesian Probabilistic Annotation of LCMS-Based Metabolomics.” <em>Bioinformatics</em> 30 (9): 1336–7. doi:<a href="https://doi.org/10.1093/bioinformatics/btu019">10.1093/bioinformatics/btu019</a>.</p>
</div>
<div>
<p>Siskos, Alexandros P., Pooja Jain, Werner Römisch-Margl, Mark Bennett, David Achaintre, Yasmin Asad, Luke Marney, et al. 2017. “Interlaboratory Reproducibility of a Targeted Metabolomics Platform for Analysis of Human Serum and Plasma.” <em>Anal. Chem.</em> 89 (1): 656–65. doi:<a href="https://doi.org/10.1021/acs.analchem.6b02930">10.1021/acs.analchem.6b02930</a>.</p>
</div>
<div>
<p>Sitnikov, Dmitri G., Cian S. Monnin, and Dajana Vuckovic. 2016. “Systematic Assessment of Seven Solvent and Solid-Phase Extraction Methods for Metabolomics Analysis of Human Plasma by LC-MS.” <em>Sci Rep</em> 6 (December). doi:<a href="https://doi.org/10.1038/srep38885">10.1038/srep38885</a>.</p>
</div>
<div>
<p>Smirnov, Kirill S., Tanja V. Maier, Alesia Walker, Silke S. Heinzmann, Sara Forcisi, Inés Martinez, Jens Walter, and Philippe Schmitt-Kopplin. 2016. “Challenges of Metabolomics in Human Gut Microbiota Research.” <em>International Journal of Medical Microbiology</em>, Intestinal microbiota - a microbial ecosystem at the edge between immune homeostasis and inflammation, 306 (5): 266–79. doi:<a href="https://doi.org/10.1016/j.ijmm.2016.03.006">10.1016/j.ijmm.2016.03.006</a>.</p>
</div>
<div>
<p>Smith, Colin A., Elizabeth J. Want, Grace O’Maille, Ruben Abagyan, and Gary Siuzdak. 2006. “XCMS: Processing Mass Spectrometry Data for Metabolite Profiling Using Nonlinear Peak Alignment, Matching, and Identification.” <em>Anal. Chem.</em> 78 (3): 779–87. doi:<a href="https://doi.org/10.1021/ac051437y">10.1021/ac051437y</a>.</p>
</div>
<div>
<p>Spicer, Rachel, Reza M. Salek, Pablo Moreno, Daniel Cañueto, and Christoph Steinbeck. 2017. “Navigating Freely-Available Software Tools for Metabolomics Analysis.” <em>Metabolomics</em> 13 (9). doi:<a href="https://doi.org/10.1007/s11306-017-1242-7">10.1007/s11306-017-1242-7</a>.</p>
</div>
<div>
<p>Spratlin, Jennifer L., Natalie J. Serkova, and S. Gail Eckhardt. 2009. “Clinical Applications of Metabolomics in Oncology: A Review.” <em>Clin Cancer Res</em> 15 (2): 431–40. doi:<a href="https://doi.org/10.1158/1078-0432.CCR-08-1059">10.1158/1078-0432.CCR-08-1059</a>.</p>
</div>
<div>
<p>Sumner, Lloyd W, Pedro Mendes, and Richard A Dixon. 2003. “Plant Metabolomics: Large-Scale Phytochemistry in the Functional Genomics Era.” <em>Phytochemistry</em>, Plant metabolomics, 62 (6): 817–36. doi:<a href="https://doi.org/10.1016/S0031-9422(02)00708-2">10.1016/S0031-9422(02)00708-2</a>.</p>
</div>
<div>
<p>Sumner, Lloyd W., Alexander Amberg, Dave Barrett, Michael H. Beale, Richard Beger, Clare A. Daykin, Teresa W.-M. Fan, et al. 2007. “Proposed Minimum Reporting Standards for Chemical Analysis Chemical Analysis Working Group (CAWG) Metabolomics Standards Initiative (MSI).” <em>Metabolomics</em> 3 (3): 211–21. doi:<a href="https://doi.org/10.1007/s11306-007-0082-2">10.1007/s11306-007-0082-2</a>.</p>
</div>
<div>
<p>Tan, S. Z., P. Begley, G. Mullard, K. A. Hollywood, and P. N. Bishop. 2016. “Introduction to Metabolomics and Its Applications in Ophthalmology.” <em>Eye</em> 30 (6): 773–83. doi:<a href="https://doi.org/10.1038/eye.2016.37">10.1038/eye.2016.37</a>.</p>
</div>
<div>
<p>Tautenhahn, Ralf, Christoph Böttcher, and Steffen Neumann. 2008. “Highly Sensitive Feature Detection for High Resolution LC/MS.” <em>BMC Bioinformatics</em> 9: 504. doi:<a href="https://doi.org/10.1186/1471-2105-9-504">10.1186/1471-2105-9-504</a>.</p>
</div>
<div>
<p>Theodoridis, Georgios A., Helen G. Gika, Elizabeth J. Want, and Ian D. Wilson. 2012. “Liquid ChromatographyMass Spectrometry Based Global Metabolite Profiling: A Review.” <em>Analytica Chimica Acta</em> 711 (January): 7–16. doi:<a href="https://doi.org/10.1016/j.aca.2011.09.042">10.1016/j.aca.2011.09.042</a>.</p>
</div>
<div>
<p>Thonusin, Chanisa, Heidi B. IglayReger, Tanu Soni, Amy E. Rothberg, Charles F. Burant, and Charles R. Evans. 2017. “Evaluation of Intensity Drift Correction Strategies Using MetaboDrift, a Normalization Tool for Multi-Batch Metabolomics Data.” <em>Journal of Chromatography A</em>, Pushing the boundaries of chromatography and electrophoresis, 1523 (Supplement C): 265–74. doi:<a href="https://doi.org/10.1016/j.chroma.2017.09.023">10.1016/j.chroma.2017.09.023</a>.</p>
</div>
<div>
<p>Tian, Tze-Feng, San-Yuan Wang, Tien-Chueh Kuo, Cheng-En Tan, Guan-Yuan Chen, Ching-Hua Kuo, Chi-Hsin Sally Chen, Chang-Chuan Chan, Olivia A. Lin, and Y. Jane Tseng. 2016. “Web Server for Peak Detection, Baseline Correction, and Alignment in Two-Dimensional Gas Chromatography Mass Spectrometry-Based Metabolomics Data.” <em>Anal. Chem.</em> 88 (21): 10395–10403. doi:<a href="https://doi.org/10.1021/acs.analchem.6b00755">10.1021/acs.analchem.6b00755</a>.</p>
</div>
<div>
<p>Treutler, Hendrik, Hiroshi Tsugawa, Andrea Porzel, Karin Gorzolka, Alain Tissier, Steffen Neumann, and Gerd Ulrich Balcke. 2016. “Discovering Regulated Metabolite Families in Untargeted Metabolomics Studies.” <em>Anal. Chem.</em> 88 (16): 8082–90. doi:<a href="https://doi.org/10.1021/acs.analchem.6b01569">10.1021/acs.analchem.6b01569</a>.</p>
</div>
<div>
<p>Tsou, Chih-Chiang, Dmitry Avtonomov, Brett Larsen, Monika Tucholska, Hyungwon Choi, Anne-Claude Gingras, and Alexey I. Nesvizhskii. 2015. “DIA-Umpire: Comprehensive Computational Framework for Data-Independent Acquisition Proteomics.” <em>Nat. Methods</em> 12 (3): 258–64. doi:<a href="https://doi.org/10.1038/nmeth.3255">10.1038/nmeth.3255</a>.</p>
</div>
<div>
<p>Tsugawa, Hiroshi, Tomas Cajka, Tobias Kind, Yan Ma, Brendan Higgins, Kazutaka Ikeda, Mitsuhiro Kanazawa, Jean VanderGheynst, Oliver Fiehn, and Masanori Arita. 2015. “MS-DIAL: Data-Independent MS/MS Deconvolution for Comprehensive Metabolome Analysis.” <em>Nat Meth</em> 12 (6): 523–26. doi:<a href="https://doi.org/10.1038/nmeth.3393">10.1038/nmeth.3393</a>.</p>
</div>
<div>
<p>Tsugawa, Hiroshi, Tobias Kind, Ryo Nakabayashi, Daichi Yukihira, Wataru Tanaka, Tomas Cajka, Kazuki Saito, Oliver Fiehn, and Masanori Arita. 2016. “Hydrogen Rearrangement Rules: Computational MS/MS Fragmentation and Structure Elucidation Using MS-FINDER Software.” <em>Anal. Chem.</em> 88 (16): 7946–58. doi:<a href="https://doi.org/10.1021/acs.analchem.6b00770">10.1021/acs.analchem.6b00770</a>.</p>
</div>
<div>
<p>Tumas, Jaroslav, Kotryna Kvederaviciute, Marius Petrulionis, Benediktas Kurlinkus, Arnas Rimkus, Greta Sakalauskaite, Jonas Cicenas, and Audrius Sileikis. 2016. “Metabolomics in Pancreatic Cancer Biomarkers Research.” <em>Med Oncol</em> 33 (12): 133. doi:<a href="https://doi.org/10.1007/s12032-016-0853-6">10.1007/s12032-016-0853-6</a>.</p>
</div>
<div>
<p>Uppal, Karan, Quinlyn A. Soltow, Frederick H. Strobel, W. Stephen Pittard, Kim M. Gernert, Tianwei Yu, and Dean P. Jones. 2013. “XMSanalyzer: Automated Pipeline for Improved Feature Detection and Downstream Analysis of Large-Scale, Non-Targeted Metabolomics Data.” <em>BMC Bioinformatics</em> 14 (1): 15. doi:<a href="https://doi.org/10.1186/1471-2105-14-15">10.1186/1471-2105-14-15</a>.</p>
</div>
<div>
<p>Uppal, Karan, Douglas I. Walker, and Dean P. Jones. 2017. “XMSannotator: An R Package for Network-Based Annotation of High-Resolution Metabolomics Data.” <em>Anal. Chem.</em> 89 (2): 1063–7. doi:<a href="https://doi.org/10.1021/acs.analchem.6b01214">10.1021/acs.analchem.6b01214</a>.</p>
</div>
<div>
<p>Uppal, Karan, Douglas I. Walker, Ken Liu, Shuzhao Li, Young-Mi Go, and Dean P. Jones. 2016. “Computational Metabolomics: A Framework for the Million Metabolome.” <em>Chem. Res. Toxicol.</em> 29 (12): 1956–75. doi:<a href="https://doi.org/10.1021/acs.chemrestox.6b00179">10.1021/acs.chemrestox.6b00179</a>.</p>
</div>
<div>
<p>Viant, Mark R, Irwin J Kurland, Martin R Jones, and Warwick B Dunn. 2017. “How Close Are We to Complete Annotation of Metabolomes?” <em>Current Opinion in Chemical Biology</em>, Omics, 36 (February): 64–69. doi:<a href="https://doi.org/10.1016/j.cbpa.2017.01.001">10.1016/j.cbpa.2017.01.001</a>.</p>
</div>
<div>
<p>Vinaixa, Maria, Emma L. Schymanski, Steffen Neumann, Miriam Navarro, Reza M. Salek, and Oscar Yanes. 2016. “Mass Spectral Databases for LC/MS- and GC/MS-Based Metabolomics: State of the Field and Future Prospects.” <em>TrAC Trends in Analytical Chemistry</em> 78 (April): 23–35. doi:<a href="https://doi.org/10.1016/j.trac.2015.09.005">10.1016/j.trac.2015.09.005</a>.</p>
</div>
<div>
<p>Wang, Mingxun, Jeremy J. Carver, Vanessa V. Phelan, Laura M. Sanchez, Neha Garg, Yao Peng, Don Duy Nguyen, et al. 2016. “Sharing and Community Curation of Mass Spectrometry Data with Global Natural Products Social Molecular Networking.” <em>Nat. Biotechnol.</em> 34 (8): 828–37. doi:<a href="https://doi.org/10.1038/nbt.3597">10.1038/nbt.3597</a>.</p>
</div>
<div>
<p>Wang, San-Yuan, Ching-Hua Kuo, and Yufeng J. Tseng. 2013. “Batch Normalizer: A Fast Total Abundance Regression Calibration Method to Simultaneously Adjust Batch and Injection Order Effects in Liquid Chromatography/Time-of-Flight Mass Spectrometry-Based Metabolomics Data and Comparison with Current Calibration Methods.” <em>Anal. Chem.</em> 85 (2): 1037–46. doi:<a href="https://doi.org/10.1021/ac302877x">10.1021/ac302877x</a>.</p>
</div>
<div>
<p>Warth, Benedikt, Scott Spangler, Mingliang Fang, Caroline H. Johnson, Erica M. Forsberg, Ana Granados, Richard L. Martin, et al. 2017. “Exposome-Scale Investigations Guided by Global Metabolomics, Pathway Analysis, and Cognitive Computing.” <em>Anal. Chem.</em> 89 (21): 11505–13. doi:<a href="https://doi.org/10.1021/acs.analchem.7b02759">10.1021/acs.analchem.7b02759</a>.</p>
</div>
<div>
<p>Weber, Ralf J. M., and Mark R. Viant. 2010. “MI-Pack: Increased Confidence of Metabolite Identification in Mass Spectra by Integrating Accurate Masses and Metabolic Pathways.” <em>Chemometrics and Intelligent Laboratory Systems</em>, OMICS, 104 (1): 75–82. doi:<a href="https://doi.org/10.1016/j.chemolab.2010.04.010">10.1016/j.chemolab.2010.04.010</a>.</p>
</div>
<div>
<p>Weljie, Aalim M., Jack Newton, Pascal Mercier, Erin Carlson, and Carolyn M. Slupsky. 2006. “Targeted Profiling: Quantitative Analysis of 1H NMR Metabolomics Data.” <em>Anal. Chem.</em> 78 (13): 4430–42. doi:<a href="https://doi.org/10.1021/ac060209g">10.1021/ac060209g</a>.</p>
</div>
<div>
<p>Wiklund, Susanne, Erik Johansson, Lina Sjöström, Ewa J. Mellerowicz, Ulf Edlund, John P. Shockcor, Johan Gottfries, Thomas Moritz, and Johan Trygg. 2008. “Visualization of GC/TOF-MS-Based Metabolomics Data for Identification of Biochemically Interesting Compounds Using OPLS Class Models.” <em>Anal. Chem.</em> 80 (1): 115–22. doi:<a href="https://doi.org/10.1021/ac0713510">10.1021/ac0713510</a>.</p>
</div>
<div>
<p>Wishart, David S. 2016. “Emerging Applications of Metabolomics in Drug Discovery and Precision Medicine.” <em>Nat Rev Drug Discov</em> 15 (7): 473–84. doi:<a href="https://doi.org/10.1038/nrd.2016.32">10.1038/nrd.2016.32</a>.</p>
</div>
<div>
<p>Witting, Michael, Christoph Ruttkies, Steffen Neumann, and Philippe Schmitt-Kopplin. 2017. “LipidFrag: Improving Reliability of in Silico Fragmentation of Lipids and Application to the Caenorhabditis Elegans Lipidome.” <em>PLOS ONE</em> 12 (3): e0172311. doi:<a href="https://doi.org/10.1371/journal.pone.0172311">10.1371/journal.pone.0172311</a>.</p>
</div>
<div>
<p>Wu, Yiman, and Liang Li. 2016. “Sample Normalization Methods in Quantitative Metabolomics.” <em>Journal of Chromatography A</em>, Editors’ choice x, 1430 (January): 80–95. doi:<a href="https://doi.org/10.1016/j.chroma.2015.12.007">10.1016/j.chroma.2015.12.007</a>.</p>
</div>
<div>
<p>Yamamoto, Hiroyuki, Tamaki Fujimori, Hajime Sato, Gen Ishikawa, Kenjiro Kami, and Yoshiaki Ohashi. 2014. “Statistical Hypothesis Testing of Factor Loading in Principal Component Analysis and Its Application to Metabolite Set Enrichment Analysis.” <em>BMC Bioinformatics</em> 15 (February): 51. doi:<a href="https://doi.org/10.1186/1471-2105-15-51">10.1186/1471-2105-15-51</a>.</p>
</div>
<div>
<p>Yang, Qin, Shan-Shan Lin, Jiang-Tao Yang, Li-Juan Tang, and Ru-Qin Yu. 2017. “Detection of Inborn Errors of Metabolism Utilizing GC-MS Urinary Metabolomics Coupled with a Modified Orthogonal Partial Least Squares Discriminant Analysis.” <em>Talanta</em> 165 (April): 545–52. doi:<a href="https://doi.org/10.1016/j.talanta.2017.01.018">10.1016/j.talanta.2017.01.018</a>.</p>
</div>
<div>
<p>Yu, Miao, Mariola Olkowicz, and Janusz Pawliszyn. 2018. “Structure/Reaction Directed Analysis for LC-MS Based Untargeted Analysis.” <em>Analytica Chimica Acta</em>, October. doi:<a href="https://doi.org/10.1016/j.aca.2018.10.062">10.1016/j.aca.2018.10.062</a>.</p>
</div>
<div>
<p>Yu, Tianwei, Youngja Park, Jennifer M. Johnson, and Dean P. Jones. 2009. “ApLCMSAdaptive Processing of High-Resolution LC/MS Data.” <em>Bioinformatics</em> 25 (15): 1930–6. doi:<a href="https://doi.org/10.1093/bioinformatics/btp291">10.1093/bioinformatics/btp291</a>.</p>
</div>
<div>
<p>Yuan, Min, Susanne B. Breitkopf, Xuemei Yang, and John M. Asara. 2012. “A Positive/Negative IonSwitching, Targeted Mass SpectrometryBased Metabolomics Platform for Bodily Fluids, Cells, and Fresh and Fixed Tissue.” <em>Nat. Protoc.</em> 7 (5): 872–81. doi:<a href="https://doi.org/10.1038/nprot.2012.024">10.1038/nprot.2012.024</a>.</p>
</div>
<div>
<p>Zampieri, Mattia, Karthik Sekar, Nicola Zamboni, and Uwe Sauer. 2017. “Frontiers of High-Throughput Metabolomics.” <em>Current Opinion in Chemical Biology</em>, Omics, 36 (February): 15–23. doi:<a href="https://doi.org/10.1016/j.cbpa.2016.12.006">10.1016/j.cbpa.2016.12.006</a>.</p>
</div>
<div>
<p>Zenobi, R. 2013. “Single-Cell Metabolomics: Analytical and Biological Perspectives.” <em>Science</em> 342 (6163): 1243259. doi:<a href="https://doi.org/10.1126/science.1243259">10.1126/science.1243259</a>.</p>
</div>
<div>
<p>Zha, Haihong, Yuping Cai, Yandong Yin, Zhuozhong Wang, Kang Li, and Zheng-Jiang Zhu. 2018. “SWATHtoMRM: Development of High-Coverage Targeted Metabolomics Method Using SWATH Technology for Biomarker Discovery.” <em>Anal. Chem.</em> 90 (6): 4062–70. doi:<a href="https://doi.org/10.1021/acs.analchem.7b05318">10.1021/acs.analchem.7b05318</a>.</p>
</div>
<div>
<p>Zhang, Aihua, Hui Sun, Ping Wang, Ying Han, and Xijun Wang. 2012. “Modern Analytical Techniques in Metabolomics Analysis.” <em>The Analyst</em> 137 (2): 293–300. doi:<a href="https://doi.org/10.1039/C1AN15605E">10.1039/C1AN15605E</a>.</p>
</div>
<div>
<p>Zhou, Juntuo, and Yuxin Yin. 2016. “Strategies for Large-Scale Targeted Metabolomics Quantification by Liquid Chromatography-Mass Spectrometry.” <em>Analyst</em> 141 (23): 6362–73. doi:<a href="https://doi.org/10.1039/C6AN01753C">10.1039/C6AN01753C</a>.</p>
</div>
</div>
</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-mannhold2009">
<p>Mannhold, Raimund, Gennadiy I. Poda, Claude Ostermann, and Igor V. Tetko. 2009. “Calculation of Molecular Lipophilicity: State-of-the-Art and Comparison of LogP Methods on More Than 96,000 Compounds.” <em>Journal of Pharmaceutical Sciences</em> 98 (3): 861–93. doi:<a href="https://doi.org/10.1002/jps.21494">10.1002/jps.21494</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="demo-1.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/yufree/metaworkflow/edit/master/11-exposome.Rmd",
"text": "Edit"
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
