<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Meta-Workflow</title>
  <meta name="description" content="This is a workflow for metabolomics studies.">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="Meta-Workflow" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="images/cover.png" />
  <meta property="og:description" content="This is a workflow for metabolomics studies." />
  <meta name="github-repo" content="yufree/metaworkflow" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Meta-Workflow" />
  
  <meta name="twitter:description" content="This is a workflow for metabolomics studies." />
  <meta name="twitter:image" content="images/cover.png" />

<meta name="author" content="Miao YU">


<meta name="date" content="2018-10-15">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="index.html">
<link rel="next" href="exprimental-designdoe.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.3/htmlwidgets.js"></script>
<script src="libs/viz-0.3/viz.js"></script>
<link href="libs/DiagrammeR-styles-0.2/styles.css" rel="stylesheet" />
<script src="libs/grViz-binding-1.0.0/grViz.js"></script>


<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Metabolomics Workflow</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#history"><i class="fa fa-check"></i><b>1.1</b> History</a><ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#history-of-mass-spectrometry"><i class="fa fa-check"></i><b>1.1.1</b> History of Mass Spectrometry</a></li>
<li class="chapter" data-level="1.1.2" data-path="introduction.html"><a href="introduction.html#history-of-metabolomcis"><i class="fa fa-check"></i><b>1.1.2</b> History of Metabolomcis</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#reviews-and-tutorials"><i class="fa fa-check"></i><b>1.2</b> Reviews and tutorials</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#trends-in-metabolomics"><i class="fa fa-check"></i><b>1.3</b> Trends in Metabolomics</a><ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#quantitative-metabolomics"><i class="fa fa-check"></i><b>1.3.1</b> Quantitative Metabolomics</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#high-throughput-metabolomics"><i class="fa fa-check"></i><b>1.3.2</b> High throughput Metabolomics</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#workflow"><i class="fa fa-check"></i><b>1.4</b> Workflow</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="exprimental-designdoe.html"><a href="exprimental-designdoe.html"><i class="fa fa-check"></i><b>2</b> Exprimental design(DoE)</a><ul>
<li class="chapter" data-level="2.1" data-path="exprimental-designdoe.html"><a href="exprimental-designdoe.html#software"><i class="fa fa-check"></i><b>2.1</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="pretreatment.html"><a href="pretreatment.html"><i class="fa fa-check"></i><b>3</b> Pretreatment</a><ul>
<li class="chapter" data-level="3.1" data-path="pretreatment.html"><a href="pretreatment.html#quenching"><i class="fa fa-check"></i><b>3.1</b> Quenching</a></li>
<li class="chapter" data-level="3.2" data-path="pretreatment.html"><a href="pretreatment.html#extraction"><i class="fa fa-check"></i><b>3.2</b> Extraction</a></li>
<li class="chapter" data-level="3.3" data-path="pretreatment.html"><a href="pretreatment.html#instrumental-analysis"><i class="fa fa-check"></i><b>3.3</b> Instrumental analysis</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html"><i class="fa fa-check"></i><b>4</b> Raw data pretreatment</a><ul>
<li class="chapter" data-level="4.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#peak-extraction"><i class="fa fa-check"></i><b>4.1</b> Peak extraction</a></li>
<li class="chapter" data-level="4.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#retention-time-correction"><i class="fa fa-check"></i><b>4.2</b> Retention Time Correction</a></li>
<li class="chapter" data-level="4.3" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#filling-missing-values"><i class="fa fa-check"></i><b>4.3</b> Filling missing values</a></li>
<li class="chapter" data-level="4.4" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#spectral-deconvolution"><i class="fa fa-check"></i><b>4.4</b> Spectral deconvolution</a></li>
<li class="chapter" data-level="4.5" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#dynamic-range"><i class="fa fa-check"></i><b>4.5</b> Dynamic Range</a><ul>
<li class="chapter" data-level="4.5.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#non-detects"><i class="fa fa-check"></i><b>4.5.1</b> Non-detects</a></li>
<li class="chapter" data-level="4.5.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#over-detection-limit"><i class="fa fa-check"></i><b>4.5.2</b> Over Detection Limit</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#rsd-filter"><i class="fa fa-check"></i><b>4.6</b> RSD Filter</a></li>
<li class="chapter" data-level="4.7" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#power-analysis-filter"><i class="fa fa-check"></i><b>4.7</b> Power Analysis Filter</a></li>
<li class="chapter" data-level="4.8" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#normalization"><i class="fa fa-check"></i><b>4.8</b> Normalization</a><ul>
<li class="chapter" data-level="4.8.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#unsupervised-methods"><i class="fa fa-check"></i><b>4.8.1</b> Unsupervised methods</a></li>
<li class="chapter" data-level="4.8.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#supervised-methods"><i class="fa fa-check"></i><b>4.8.2</b> Supervised methods</a></li>
</ul></li>
<li class="chapter" data-level="4.9" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#software-1"><i class="fa fa-check"></i><b>4.9</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="peaks-selection.html"><a href="peaks-selection.html"><i class="fa fa-check"></i><b>5</b> Peaks selection</a><ul>
<li class="chapter" data-level="5.1" data-path="peaks-selection.html"><a href="peaks-selection.html#peak-misidentification"><i class="fa fa-check"></i><b>5.1</b> Peak misidentification</a></li>
<li class="chapter" data-level="5.2" data-path="peaks-selection.html"><a href="peaks-selection.html#software-2"><i class="fa fa-check"></i><b>5.2</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="annotation.html"><a href="annotation.html"><i class="fa fa-check"></i><b>6</b> Annotation</a><ul>
<li class="chapter" data-level="6.1" data-path="annotation.html"><a href="annotation.html#issues-in-annotation"><i class="fa fa-check"></i><b>6.1</b> Issues in annotation</a></li>
<li class="chapter" data-level="6.2" data-path="annotation.html"><a href="annotation.html#annotation-v.s.-identification"><i class="fa fa-check"></i><b>6.2</b> Annotation v.s. identification</a></li>
<li class="chapter" data-level="6.3" data-path="annotation.html"><a href="annotation.html#ms-database-for-annotation"><i class="fa fa-check"></i><b>6.3</b> MS Database for annotation</a><ul>
<li class="chapter" data-level="6.3.1" data-path="annotation.html"><a href="annotation.html#msms"><i class="fa fa-check"></i><b>6.3.1</b> MS/MS</a></li>
<li class="chapter" data-level="6.3.2" data-path="annotation.html"><a href="annotation.html#ms"><i class="fa fa-check"></i><b>6.3.2</b> MS</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="annotation.html"><a href="annotation.html#compounds-database"><i class="fa fa-check"></i><b>6.4</b> Compounds Database</a></li>
<li class="chapter" data-level="6.5" data-path="annotation.html"><a href="annotation.html#software-3"><i class="fa fa-check"></i><b>6.5</b> Software</a><ul>
<li class="chapter" data-level="6.5.1" data-path="annotation.html"><a href="annotation.html#adducts-list"><i class="fa fa-check"></i><b>6.5.1</b> Adducts list</a></li>
<li class="chapter" data-level="6.5.2" data-path="annotation.html"><a href="annotation.html#camera"><i class="fa fa-check"></i><b>6.5.2</b> CAMERA</a></li>
<li class="chapter" data-level="6.5.3" data-path="annotation.html"><a href="annotation.html#ramclustr"><i class="fa fa-check"></i><b>6.5.3</b> RAMClustR</a></li>
<li class="chapter" data-level="6.5.4" data-path="annotation.html"><a href="annotation.html#nontarget"><i class="fa fa-check"></i><b>6.5.4</b> nontarget</a></li>
<li class="chapter" data-level="6.5.5" data-path="annotation.html"><a href="annotation.html#xmsannotator"><i class="fa fa-check"></i><b>6.5.5</b> xMSannotator</a></li>
<li class="chapter" data-level="6.5.6" data-path="annotation.html"><a href="annotation.html#mine"><i class="fa fa-check"></i><b>6.5.6</b> MINE</a></li>
<li class="chapter" data-level="6.5.7" data-path="annotation.html"><a href="annotation.html#interpretmsspectrum"><i class="fa fa-check"></i><b>6.5.7</b> InterpretMSSpectrum</a></li>
<li class="chapter" data-level="6.5.8" data-path="annotation.html"><a href="annotation.html#for-ident"><i class="fa fa-check"></i><b>6.5.8</b> For Ident</a></li>
<li class="chapter" data-level="6.5.9" data-path="annotation.html"><a href="annotation.html#mzmatch"><i class="fa fa-check"></i><b>6.5.9</b> mzmatch</a></li>
<li class="chapter" data-level="6.5.10" data-path="annotation.html"><a href="annotation.html#mz.unity"><i class="fa fa-check"></i><b>6.5.10</b> mz.unity</a></li>
<li class="chapter" data-level="6.5.11" data-path="annotation.html"><a href="annotation.html#mait"><i class="fa fa-check"></i><b>6.5.11</b> MAIT</a></li>
<li class="chapter" data-level="6.5.12" data-path="annotation.html"><a href="annotation.html#probmetab"><i class="fa fa-check"></i><b>6.5.12</b> ProbMetab</a></li>
<li class="chapter" data-level="6.5.13" data-path="annotation.html"><a href="annotation.html#ramsi"><i class="fa fa-check"></i><b>6.5.13</b> RAMSI</a></li>
<li class="chapter" data-level="6.5.14" data-path="annotation.html"><a href="annotation.html#sirius"><i class="fa fa-check"></i><b>6.5.14</b> Sirius</a></li>
<li class="chapter" data-level="6.5.15" data-path="annotation.html"><a href="annotation.html#mi-pack"><i class="fa fa-check"></i><b>6.5.15</b> MI-Pack</a></li>
<li class="chapter" data-level="6.5.16" data-path="annotation.html"><a href="annotation.html#plantmat"><i class="fa fa-check"></i><b>6.5.16</b> Plantmat</a></li>
<li class="chapter" data-level="6.5.17" data-path="annotation.html"><a href="annotation.html#metfamily"><i class="fa fa-check"></i><b>6.5.17</b> MetFamily</a></li>
<li class="chapter" data-level="6.5.18" data-path="annotation.html"><a href="annotation.html#lipidmatch"><i class="fa fa-check"></i><b>6.5.18</b> Lipidmatch</a></li>
<li class="chapter" data-level="6.5.19" data-path="annotation.html"><a href="annotation.html#molfind"><i class="fa fa-check"></i><b>6.5.19</b> MolFind</a></li>
<li class="chapter" data-level="6.5.20" data-path="annotation.html"><a href="annotation.html#metfusion"><i class="fa fa-check"></i><b>6.5.20</b> MetFusion</a></li>
<li class="chapter" data-level="6.5.21" data-path="annotation.html"><a href="annotation.html#imet"><i class="fa fa-check"></i><b>6.5.21</b> iMet</a></li>
<li class="chapter" data-level="6.5.22" data-path="annotation.html"><a href="annotation.html#metscape"><i class="fa fa-check"></i><b>6.5.22</b> Metscape</a></li>
<li class="chapter" data-level="6.5.23" data-path="annotation.html"><a href="annotation.html#metfrag"><i class="fa fa-check"></i><b>6.5.23</b> MetFrag</a></li>
<li class="chapter" data-level="6.5.24" data-path="annotation.html"><a href="annotation.html#lipidfrag"><i class="fa fa-check"></i><b>6.5.24</b> LipidFrag</a></li>
<li class="chapter" data-level="6.5.25" data-path="annotation.html"><a href="annotation.html#mycompoundid"><i class="fa fa-check"></i><b>6.5.25</b> MycompoundID</a></li>
<li class="chapter" data-level="6.5.26" data-path="annotation.html"><a href="annotation.html#cfm-id"><i class="fa fa-check"></i><b>6.5.26</b> CFM-ID</a></li>
<li class="chapter" data-level="6.5.27" data-path="annotation.html"><a href="annotation.html#magma"><i class="fa fa-check"></i><b>6.5.27</b> magma</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="omics-analysis.html"><a href="omics-analysis.html"><i class="fa fa-check"></i><b>7</b> Omics analysis</a><ul>
<li class="chapter" data-level="7.1" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-analysis"><i class="fa fa-check"></i><b>7.1</b> Pathway analysis</a><ul>
<li class="chapter" data-level="7.1.1" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-database"><i class="fa fa-check"></i><b>7.1.1</b> Pathway Database</a></li>
<li class="chapter" data-level="7.1.2" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-software"><i class="fa fa-check"></i><b>7.1.2</b> Pathway software</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="omics-analysis.html"><a href="omics-analysis.html#network-analysis"><i class="fa fa-check"></i><b>7.2</b> Network analysis</a></li>
<li class="chapter" data-level="7.3" data-path="omics-analysis.html"><a href="omics-analysis.html#omics-integration"><i class="fa fa-check"></i><b>7.3</b> Omics integration</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html"><i class="fa fa-check"></i><b>8</b> Common analysis methods for metabolomics</a><ul>
<li class="chapter" data-level="8.1" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#pca"><i class="fa fa-check"></i><b>8.1</b> PCA</a></li>
<li class="chapter" data-level="8.2" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#cluster-analysis"><i class="fa fa-check"></i><b>8.2</b> Cluster Analysis</a></li>
<li class="chapter" data-level="8.3" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#plsda"><i class="fa fa-check"></i><b>8.3</b> PLSDA</a></li>
<li class="chapter" data-level="8.4" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#self-organizing-map"><i class="fa fa-check"></i><b>8.4</b> Self-organizing map</a></li>
<li class="chapter" data-level="8.5" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#canonical-correlation-analysis"><i class="fa fa-check"></i><b>8.5</b> Canonical correlation analysis</a></li>
<li class="chapter" data-level="8.6" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#softwareplatform"><i class="fa fa-check"></i><b>8.6</b> Software&amp;platform</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="demo.html"><a href="demo.html"><i class="fa fa-check"></i><b>9</b> Demo</a><ul>
<li class="chapter" data-level="9.1" data-path="demo.html"><a href="demo.html#project-setup"><i class="fa fa-check"></i><b>9.1</b> Project Setup</a></li>
<li class="chapter" data-level="9.2" data-path="demo.html"><a href="demo.html#data-input"><i class="fa fa-check"></i><b>9.2</b> Data input</a></li>
<li class="chapter" data-level="9.3" data-path="demo.html"><a href="demo.html#find-the-peaks"><i class="fa fa-check"></i><b>9.3</b> Find the peaks</a></li>
<li class="chapter" data-level="9.4" data-path="demo.html"><a href="demo.html#data-correction"><i class="fa fa-check"></i><b>9.4</b> Data correction</a></li>
<li class="chapter" data-level="9.5" data-path="demo.html"><a href="demo.html#statistic-analysis"><i class="fa fa-check"></i><b>9.5</b> Statistic analysis</a></li>
<li class="chapter" data-level="9.6" data-path="demo.html"><a href="demo.html#annotation-1"><i class="fa fa-check"></i><b>9.6</b> Annotation</a></li>
<li class="chapter" data-level="9.7" data-path="demo.html"><a href="demo.html#omics-analysis-1"><i class="fa fa-check"></i><b>9.7</b> Omics analysis</a></li>
<li class="chapter" data-level="9.8" data-path="demo.html"><a href="demo.html#metaboanalyst"><i class="fa fa-check"></i><b>9.8</b> MetaboAnalyst</a></li>
<li class="chapter" data-level="9.9" data-path="demo.html"><a href="demo.html#visulizing-peaks"><i class="fa fa-check"></i><b>9.9</b> Visulizing Peaks</a></li>
<li class="chapter" data-level="9.10" data-path="demo.html"><a href="demo.html#optimation-of-xcms"><i class="fa fa-check"></i><b>9.10</b> Optimation of XCMS</a></li>
<li class="chapter" data-level="9.11" data-path="demo.html"><a href="demo.html#summary"><i class="fa fa-check"></i><b>9.11</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html"><i class="fa fa-check"></i><b>10</b> Software/Web Application/Workflow</a><ul>
<li class="chapter" data-level="10.1" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#platform-for-metabolomics"><i class="fa fa-check"></i><b>10.1</b> Platform for metabolomics</a><ul>
<li class="chapter" data-level="10.1.1" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#xcms-online"><i class="fa fa-check"></i><b>10.1.1</b> XCMS online</a></li>
<li class="chapter" data-level="10.1.2" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#prime"><i class="fa fa-check"></i><b>10.1.2</b> PRIMe</a></li>
<li class="chapter" data-level="10.1.3" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#openms"><i class="fa fa-check"></i><b>10.1.3</b> OpenMS</a></li>
<li class="chapter" data-level="10.1.4" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#mzmine-2"><i class="fa fa-check"></i><b>10.1.4</b> MZmine 2</a></li>
<li class="chapter" data-level="10.1.5" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#xcms"><i class="fa fa-check"></i><b>10.1.5</b> XCMS</a></li>
<li class="chapter" data-level="10.1.6" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#emory-mahpic"><i class="fa fa-check"></i><b>10.1.6</b> Emory MaHPIC</a></li>
<li class="chapter" data-level="10.1.7" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#dia-data-analysis"><i class="fa fa-check"></i><b>10.1.7</b> DIA data analysis</a></li>
<li class="chapter" data-level="10.1.8" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#others"><i class="fa fa-check"></i><b>10.1.8</b> Others</a></li>
</ul></li>
<li class="chapter" data-level="10.2" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#data-sharing"><i class="fa fa-check"></i><b>10.2</b> Data sharing</a></li>
<li class="chapter" data-level="10.3" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#contest"><i class="fa fa-check"></i><b>10.3</b> Contest</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="case-studyselected.html"><a href="case-studyselected.html"><i class="fa fa-check"></i><b>11</b> Case Study(selected)</a><ul>
<li class="chapter" data-level="11.1" data-path="case-studyselected.html"><a href="case-studyselected.html#cancer"><i class="fa fa-check"></i><b>11.1</b> Cancer</a></li>
<li class="chapter" data-level="11.2" data-path="case-studyselected.html"><a href="case-studyselected.html#mental-health"><i class="fa fa-check"></i><b>11.2</b> Mental Health</a></li>
<li class="chapter" data-level="11.3" data-path="case-studyselected.html"><a href="case-studyselected.html#interesting-papers"><i class="fa fa-check"></i><b>11.3</b> Interesting papers</a></li>
<li class="chapter" data-level="11.4" data-path="case-studyselected.html"><a href="case-studyselected.html#environmental-pulltions"><i class="fa fa-check"></i><b>11.4</b> Environmental pulltions</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="exposome.html"><a href="exposome.html"><i class="fa fa-check"></i><b>12</b> Exposome</a><ul>
<li class="chapter" data-level="12.1" data-path="exposome.html"><a href="exposome.html#internal-exposure"><i class="fa fa-check"></i><b>12.1</b> Internal exposure</a></li>
<li class="chapter" data-level="12.2" data-path="exposome.html"><a href="exposome.html#external-exposure"><i class="fa fa-check"></i><b>12.2</b> External exposure</a><ul>
<li class="chapter" data-level="12.2.1" data-path="exposome.html"><a href="exposome.html#environmental-fate-of-compounds"><i class="fa fa-check"></i><b>12.2.1</b> Environmental fate of compounds</a></li>
<li class="chapter" data-level="12.2.2" data-path="exposome.html"><a href="exposome.html#exposure-study-database"><i class="fa fa-check"></i><b>12.2.2</b> Exposure study database</a></li>
</ul></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/yufree/metaworkflow" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Meta-Workflow</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="introduction" class="section level1">
<h1><span class="header-section-number">Chapter 1</span> Introduction</h1>
<p>Information in living organism commuicates along the Genomics, Transcriptomics, Proteomics and Metabolomics in Central dogma. Following such stream, we might answer certain problems in different scales from individual, population, community to ecosystem. Metabolomics (i.e., the profiling and quantitation of metabolites in body fluids) is a relatively new field of “omics” studies. Different from other omics studies, metabolomics always focused on small moleculars with much lower mass than polypeptide with single or doubled charged ions. Metabolomics studies are always performed in GC-MS, GC*GC-MS<span class="citation">(Tian et al. <a href="#ref-tian2016">2016</a>)</span>, LC-MS, LC-MS/MS or NMR. This workflow would only cover mass spectrometry based metabolomics or XC-MS based research.</p>
<div id="history" class="section level2">
<h2><span class="header-section-number">1.1</span> History</h2>
<div id="history-of-mass-spectrometry" class="section level3">
<h3><span class="header-section-number">1.1.1</span> History of Mass Spectrometry</h3>
<ul>
<li>1913, Sir Joseph John Thomson “Rays of Positive Electricity and Their Application to Chemical Analyses.”</li>
</ul>
<div class="figure"><span id="fig:history"></span>
<img src="images/mshistory.jpg" alt="Sir Joseph John Thomson &quot;Rays of Positive Electricity and Their Application to Chemical Analyses.&quot;"  />
<p class="caption">
Figure 1.1: Sir Joseph John Thomson “Rays of Positive Electricity and Their Application to Chemical Analyses.”
</p>
</div>
<ul>
<li><p>Petroleum industry bring mass spectrometry from physics to chemistry</p></li>
<li><p>The first commercial mass spectrometer is from Consolidated Engineering Corp to analysis simple gas mixtures from petroleum</p></li>
<li><p>In World War II, U.S. use mass spectrometer to separate and enrich isotopes of uranium in Manhattan Project</p></li>
<li><p>U.S. also use mass spectrometer for organic compounds during wartime and extend the application of mass spectrometer</p></li>
<li><p>1946, TOF, William E. Stephens</p></li>
<li><p>1970s, quadrupole mass analyzer</p></li>
<li><p>1970s, R. Graham Cooks developed mass-analyzed ion kinetic energy spectrometry, or MIKES to make MRM analysis for multi-stage mass sepctrometry</p></li>
<li><p>1980s, MALDI rescue TOF and mass spectrometry move into biological application</p></li>
<li><p>1990s, Orbitrap mass spectrometry</p></li>
<li><p>2010s, Aperture Coding mass spectrometry</p></li>
</ul>
</div>
<div id="history-of-metabolomcis" class="section level3">
<h3><span class="header-section-number">1.1.2</span> History of Metabolomcis</h3>
<p>According to this book section<span class="citation">(Kusonmano, Vongsangnak, and Chumnanpuen <a href="#ref-kusonmano2016">2016</a>)</span>:</p>
<div class="figure"><span id="fig:history2"></span>
<img src="images/metahistory.jpg" alt="Metabolomics timeline during pre- and post-metabolomics era"  />
<p class="caption">
Figure 1.2: Metabolomics timeline during pre- and post-metabolomics era
</p>
</div>
<ul>
<li><p>2000-1500 BC some traditional Chinese doctors who began to evaluate the glucose level in urine of diabetic patients using ants</p></li>
<li><p>300 BC ancient Egypt and Greece that traditionally determine the urine taste to diagnose human diseases</p></li>
<li><p>1913 Joseph John Thomson and Francis William Aston mass spectrometry</p></li>
<li><p>1946 Felix Bloch and Edward Purcell Nuclear magnetic resonance</p></li>
<li><p>late 1960s chromatographic separation technique</p></li>
<li><p>1971 Pauling’s research team “Quantitative Analysis of Urine Vapor and Breath by Gas–Liquid Partition Chromatography”</p></li>
<li><p>Willmitzer and his research team pioneer group in metabolomics which suggested the promotion of the metabolomics field and its potential applications from agriculture to medicine and other related areas in the biological sciences</p></li>
<li><p>2007 Human Metabolome Project consists of databases of approximately 2500 metabolites, 1200 drugs, and 3500 food components</p></li>
<li><p>post-metabolomics era high-throughput analytical techniques</p></li>
</ul>
</div>
</div>
<div id="reviews-and-tutorials" class="section level2">
<h2><span class="header-section-number">1.2</span> Reviews and tutorials</h2>
<p>Some new reviews and tutorials related to this workflow could be found in those papers<span class="citation">(Alonso, Marsal, and Julià <a href="#ref-alonso2015">2015</a>; Cajka and Fiehn <a href="#ref-cajka2016">2016</a>; Lu and Xu <a href="#ref-lu2008">2008</a>; Schrimpe-Rutledge et al. <a href="#ref-schrimpe-rutledge2016">2016</a>; Townsend et al. <a href="#ref-townsend2016">2016</a>; Barnes et al. <a href="#ref-barnes2016">2016</a><a href="#ref-barnes2016">b</a>; Barnes et al. <a href="#ref-barnes2016a">2016</a><a href="#ref-barnes2016a">a</a>)</span>.</p>
<p>Also I noticed more and more papers showed a bunch of data process methods as strategy for metabolomics<span class="citation">(Watrous et al. <a href="#ref-watrous2017">2017</a>; Robbat Jr. et al. <a href="#ref-robbatjr.2017">2017</a>)</span>. If you only need metabolomics as tools to tell your story, such stratrgy could be a quick start for you.</p>
<p>For software, check this review<span class="citation">(Misra and van der Hooft <a href="#ref-misra2016">2016</a>)</span>.</p>
</div>
<div id="trends-in-metabolomics" class="section level2">
<h2><span class="header-section-number">1.3</span> Trends in Metabolomics</h2>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">library</span>(gtrendsR)
res &lt;-<span class="st"> </span><span class="kw">gtrends</span>(<span class="kw">c</span>(<span class="st">&quot;metabolomics&quot;</span>, <span class="st">&quot;metabolomics&quot;</span>), <span class="dt">geo =</span> <span class="kw">c</span>(<span class="st">&quot;CA&quot;</span>,<span class="st">&quot;US&quot;</span>))
<span class="kw">plot</span>(res)</code></pre></div>
<p><img src="Metabolomics_files/figure-html/gtrends-1.png" width="672" /></p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">library</span>(rentrez)
papers_by_year &lt;-<span class="st"> </span>function(years, search_term){
    <span class="kw">return</span>(<span class="kw">sapply</span>(years, function(y) <span class="kw">entrez_search</span>(<span class="dt">db=</span><span class="st">&quot;pubmed&quot;</span>,<span class="dt">term=</span>search_term, <span class="dt">mindate=</span>y, <span class="dt">maxdate=</span>y, <span class="dt">retmax=</span><span class="dv">0</span>)$count))
}
years &lt;-<span class="st"> </span><span class="dv">1987</span>:<span class="dv">2017</span>
total_papers &lt;-<span class="st"> </span><span class="kw">papers_by_year</span>(years, <span class="st">&quot;&quot;</span>)
omics &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;genomic&quot;</span>, <span class="st">&quot;epigenomic&quot;</span>, <span class="st">&quot;metabolomics&quot;</span>, <span class="st">&quot;metagenomic&quot;</span>, <span class="st">&quot;proteomic&quot;</span>, <span class="st">&quot;transcriptomic&quot;</span>, <span class="st">&quot;pharmacogenomic&quot;</span>, <span class="st">&quot;connectomic&quot;</span>)
trend_data &lt;-<span class="st"> </span><span class="kw">sapply</span>(omics, function(t) <span class="kw">papers_by_year</span>(years, t))
trend_props &lt;-<span class="st"> </span>trend_data/total_papers
<span class="kw">library</span>(reshape)
<span class="kw">library</span>(ggplot2)
trend_df &lt;-<span class="st"> </span><span class="kw">melt</span>(<span class="kw">data.frame</span>(years, trend_data), <span class="dt">id.vars=</span><span class="st">&quot;years&quot;</span>)
p &lt;-<span class="st"> </span><span class="kw">ggplot</span>(trend_df, <span class="kw">aes</span>(years, value, <span class="dt">colour=</span>variable))
p +<span class="st"> </span><span class="kw">geom_line</span>(<span class="dt">size=</span><span class="dv">1</span>) +<span class="st"> </span><span class="kw">scale_y_log10</span>(<span class="st">&quot;number of papers&quot;</span>)</code></pre></div>
<pre><code>## Warning: Transformation introduced infinite values in continuous y-axis</code></pre>
<p><img src="Metabolomics_files/figure-html/rentrez-1.png" width="672" /></p>
<div id="quantitative-metabolomics" class="section level3">
<h3><span class="header-section-number">1.3.1</span> Quantitative Metabolomics</h3>
<p>Those papers <span class="citation">(Kapoore and Vaidyanathan <a href="#ref-kapoore2016b">2016</a>; Jorge, Mata, and António <a href="#ref-jorge2016a">2016</a>)</span>.</p>
</div>
<div id="high-throughput-metabolomics" class="section level3">
<h3><span class="header-section-number">1.3.2</span> High throughput Metabolomics</h3>
<p>Those papers <span class="citation">(Zampieri et al. <a href="#ref-zampieri2017">2017</a>)</span></p>
<ul>
<li>Cohort size</li>
<li>Temporal resolution</li>
<li>Spatial resolution</li>
</ul>
</div>
</div>
<div id="workflow" class="section level2">
<h2><span class="header-section-number">1.4</span> Workflow</h2>
<div id="htmlwidget-8020276cb22f1db0e51e" style="width:672px;height:480px;" class="grViz html-widget"></div>
<script type="application/json" data-for="htmlwidget-8020276cb22f1db0e51e">{"x":{"diagram":"digraph workflow {\nnode [shape = box]\nA [label = \"Raw data\"]\nB [label = \"mzxml or CDF\"]\nC [label = \"DoE folder\"]\nD [label = \"peaks list\"]\nE [label = \"retention time correction\"]\nF [label = \"peaks grouping\"]\nG [label = \"peaks filling\"]\nH [label = \"raw peaks\"]\nI [label = \"data visulization\"]\nJ [label = \"batch effects correction\"]\nK [label = \"corrected peaks\"]\nL [label = \"annotation\"]\nM [label = \"metabolomics pathway analysis\"]\nN [label = \"omics analysis\"]\nO [label = \"biomarkers discovery/diagnoise\"]\n\nA -> B -> C -> D -> E -> F -> G -> H\nH -> I\nI -> J\nH -> J -> K -> L\nL -> M -> N\nL -> O\n                  }","config":{"engine":"dot","options":null}},"evals":[],"jsHooks":[]}</script>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-tian2016">
<p>Tian, Tze-Feng, San-Yuan Wang, Tien-Chueh Kuo, Cheng-En Tan, Guan-Yuan Chen, Ching-Hua Kuo, Chi-Hsin Sally Chen, Chang-Chuan Chan, Olivia A. Lin, and Y. Jane Tseng. 2016. “Web Server for Peak Detection, Baseline Correction, and Alignment in Two-Dimensional Gas Chromatography Mass Spectrometry-Based Metabolomics Data.” <em>Anal. Chem.</em> 88 (21): 10395–10403. doi:<a href="https://doi.org/10.1021/acs.analchem.6b00755">10.1021/acs.analchem.6b00755</a>.</p>
</div>
<div id="ref-kusonmano2016">
<p>Kusonmano, Kanthida, Wanwipa Vongsangnak, and Pramote Chumnanpuen. 2016. “Informatics for Metabolomics.” In <em>Translational Biomedical Informatics</em>, 91–115. Advances in Experimental Medicine and Biology. Springer, Singapore. doi:<a href="https://doi.org/10.1007/978-981-10-1503-8_5">10.1007/978-981-10-1503-8_5</a>.</p>
</div>
<div id="ref-alonso2015">
<p>Alonso, Arnald, Sara Marsal, and Antonio Julià. 2015. “Analytical Methods in Untargeted Metabolomics: State of the Art in 2015.” <em>Front Bioeng Biotechnol</em> 3 (March). doi:<a href="https://doi.org/10.3389/fbioe.2015.00023">10.3389/fbioe.2015.00023</a>.</p>
</div>
<div id="ref-cajka2016">
<p>Cajka, Tomas, and Oliver Fiehn. 2016. “Toward Merging Untargeted and Targeted Methods in Mass Spectrometry-Based Metabolomics and Lipidomics.” <em>Anal. Chem.</em> 88 (1): 524–45. doi:<a href="https://doi.org/10.1021/acs.analchem.5b04491">10.1021/acs.analchem.5b04491</a>.</p>
</div>
<div id="ref-lu2008">
<p>Lu, Xin, and Guowang Xu. 2008. “LC-MS Metabonomics Methodology in Biomarker Discovery.” In <em>Biomarker Methods in Drug Discovery and Development</em>, edited by Feng Wang, 291–315. Methods in Pharmacology and Toxicology. Humana Press. doi:<a href="https://doi.org/10.1007/978-1-59745-463-6_14">10.1007/978-1-59745-463-6_14</a>.</p>
</div>
<div id="ref-schrimpe-rutledge2016">
<p>Schrimpe-Rutledge, Alexandra C., Simona G. Codreanu, Stacy D. Sherrod, and John A. McLean. 2016. “Untargeted Metabolomics Strategies and Emerging Directions.” <em>J. Am. Soc. Mass Spectrom.</em> 27 (12): 1897–1905. doi:<a href="https://doi.org/10.1007/s13361-016-1469-y">10.1007/s13361-016-1469-y</a>.</p>
</div>
<div id="ref-townsend2016">
<p>Townsend, Mary K., Hugues Aschard, Immaculata De Vivo, Karin B. Michels, and Peter Kraft. 2016. “Genomics, Telomere Length, Epigenetics, and Metabolomics in the Nurses’ Health Studies.” <em>Am J Public Health</em> 106 (9): 1663–8. doi:<a href="https://doi.org/10.2105/AJPH.2016.303344">10.2105/AJPH.2016.303344</a>.</p>
</div>
<div id="ref-barnes2016">
<p>Barnes, Stephen, H. 2016b. “Training in Metabolomics Research. II. Processing and Statistical Analysis of Metabolomics Data, Metabolite Identification, Pathway Analysis, Applications of Metabolomics and Its Future.” <em>J. Mass Spectrom.</em> 51 (8): 535–48. doi:<a href="https://doi.org/10.1002/jms.3780">10.1002/jms.3780</a>.</p>
</div>
<div id="ref-barnes2016a">
<p>Barnes, Stephen, H. Paul Benton, Krista Casazza, Sara J. Cooper, Xiangqin Cui, Xiuxia Du, Jeffrey Engler, et al. 2016a. “Training in Metabolomics Research. I. Designing the Experiment, Collecting and Extracting Samples and Generating Metabolomics Data.” <em>J. Mass Spectrom.</em> 51 (7): 461–75. doi:<a href="https://doi.org/10.1002/jms.3782">10.1002/jms.3782</a>.</p>
</div>
<div id="ref-watrous2017">
<p>Watrous, Jeramie D., Mir Henglin, Brian Claggett, Kim A. Lehmann, Martin G. Larson, Susan Cheng, and Mohit Jain. 2017. “Visualization, Quantification, and Alignment of Spectral Drift in Population Scale Untargeted Metabolomics Data.” <em>Anal. Chem.</em> 89 (3): 1399–1404. doi:<a href="https://doi.org/10.1021/acs.analchem.6b04337">10.1021/acs.analchem.6b04337</a>.</p>
</div>
<div id="ref-robbatjr.2017">
<p>Robbat Jr., Albert, Nicole Kfoury, Eugene Baydakov, and Yuriy Gankin. 2017. “Optimizing Targeted/Untargeted Metabolomics by Automating Gas Chromatography/Mass Spectrometry Workflows.” <em>Journal of Chromatography A</em> 1505 (July): 96–105. doi:<a href="https://doi.org/10.1016/j.chroma.2017.05.017">10.1016/j.chroma.2017.05.017</a>.</p>
</div>
<div id="ref-misra2016">
<p>Misra, Biswapriya B., and Justin J. J. van der Hooft. 2016. “Updates in Metabolomics Tools and Resources: 20142015.” <em>ELECTROPHORESIS</em> 37 (1): 86–110. doi:<a href="https://doi.org/10.1002/elps.201500417">10.1002/elps.201500417</a>.</p>
</div>
<div id="ref-kapoore2016b">
<p>Kapoore, Rahul Vijay, and Seetharaman Vaidyanathan. 2016. “Towards Quantitative Mass Spectrometry-Based Metabolomics in Microbial and Mammalian Systems.” <em>Phil. Trans. R. Soc. A</em> 374 (2079): 20150363. doi:<a href="https://doi.org/10.1098/rsta.2015.0363">10.1098/rsta.2015.0363</a>.</p>
</div>
<div id="ref-jorge2016a">
<p>Jorge, Tiago F., Ana T. Mata, and Carla António. 2016. “Mass Spectrometry as a Quantitative Tool in Plant Metabolomics.” <em>Phil. Trans. R. Soc. A</em> 374 (2079): 20150370. doi:<a href="https://doi.org/10.1098/rsta.2015.0370">10.1098/rsta.2015.0370</a>.</p>
</div>
<div id="ref-zampieri2017">
<p>Zampieri, Mattia, Karthik Sekar, Nicola Zamboni, and Uwe Sauer. 2017. “Frontiers of High-Throughput Metabolomics.” <em>Current Opinion in Chemical Biology</em>, Omics, 36 (February): 15–23. doi:<a href="https://doi.org/10.1016/j.cbpa.2016.12.006">10.1016/j.cbpa.2016.12.006</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="exprimental-designdoe.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/yufree/metaworkflow/edit/master/01-introduction.Rmd",
"text": "Edit"
},
"download": ["Metabolomics.pdf", "Metabolomics.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
